# Age-related changes in gene expression heterogeneity in the human brain {#HeterogeneityChapter}

\small
**Declaration**

*This work includes projects I have conducted in collaboration with Veronika Kedlian, a visiting student from the Taras Shevchenko National University of Kyiv, and Ulas Isildak, an undergraduate student from the Department of Biological Sciences, Middle East Technical University [@Kedlian2019; @Isildak2020]. My contributions mostly involved the design and supervision of the study, as well as some computational analyses. I also acknowledge the contributions of my supervisor, Janet Thornton, and Mehmet Somel who contributed through fruitful discussions.*

Kedlian, V. R.\*, Donertas, H. M.\*, & Thornton, J. M. (2019). The widespread increase in inter-individual variability of gene expression in the human brain with age. Aging.
\* denotes an equal contribution

Işıldak, U., Somel, M., Thornton, J. M., & Dönertaş, H. M. (2020). Temporal changes in the gene expression heterogeneity during brain development and aging. Scientific Reports, 10(1), 4080.

**Data Availability**

*Supplementary tables and all summary statistics are available in the BioStudies database under accession number [S-BSST273](https://www.ebi.ac.uk/biostudies/studies/S-BSST273). Supplementary tables are referred with the corresponding file names throughout the text.*
\normalsize

## Introduction 

Ageing is a complex process characterised by a gradual decline in maintenance and repair mechanisms, accompanied by an increase in genetic and epigenetic mutations, and oxidative damage to nucleic acids, protein and lipids [@Gorbunova2007; @Lu2004]. The human brain experiences dramatic structural and functional changes in the course of ageing. These include decline in gray matter and white matter volumes [@Sowell2004], increase in axonal bouton dynamics [@Grillo2013] and reduced synaptic plasticity, all processes that may be associated with decline in cognitive functions [@Dorszewska2013]. Changes during brain ageing are suggested to be a result of stochastic processes, unlike changes associated with postnatal neuronal development that are known to be primarily controlled by adaptive regulatory processes [@Polleux2007; @Schratt2009; @Stefani2008]. The molecular mechanisms underlying age-related alteration of regulatory processes and eventually leading to ageing-related phenotypes, however, are little understood. 

Over the past decade, a number of transcriptome studies focusing on age-related changes in human brain gene expression profiles were published [@Lu2004; @Kang2011; @Miller2014; @Somel2010; @Tebbenkamp2014]. These studies report ageing-related differential expression patterns in many functions, including synaptic functions, energy metabolism, inflammation, stress response, and DNA repair. By analyzing age-related change in gene expression profiles in diverse brain regions, we previously showed that for many genes, gene expression changes occur in opposite directions during postnatal development (pre-20 years of age) and ageing (post-20 years of age), which may be associated with ageing-related phenotypes in healthy brain ageing [@Donertas2017]. While different brain regions are associated with specific, and often independent, gene expression profiles [@Kang2011; @Miller2014; @Tebbenkamp2014], these studies also show that age-related alteration of gene expression profiles during ageing is a widespread effect across different brain regions.

One of the suggested effects of ageing is increased heterogeneity between individuals and somatic cells, which has been previously reported by several studies. Some of these studies find an increase in age-related heterogeneity in heart, lung and white blood cells of mice [@Bahar2006; @Martinez-Jimenez2017; @Angelidis2019], *Caenorhabditis elegans* [@Herndon2002], and human twins [@Fraga2005]. A study analysing microarray datasets from different tissues of humans and rats also reported an increase in age-related heterogeneity in expression as a general trend [@Somel2006]. However, this study found no significant consistency across datasets, nor any significant enrichment in functional gene groups. That said, the generality of increase in expression heterogeneity remains unresolved. For instance, @Vinuela2018 find more decrease than an increase in heterogeneity in human twins and @Ximerakis2018 show the direction of the heterogeneity change depends on cell type in ageing mice brain. Using GTEx data covering different brain regions (20 to 70 years of age), @Brinkmeyer-Langford2016 identify a set of differentially variable genes between age groups, but they do not observe increased heterogeneity at old age. Meanwhile, another study performing single-cell RNA sequencing of human pancreatic cells, identifies an increase in transcriptional heterogeneity and somatic mutations with age [@Enge2017]. My previous research also suggested more shared expression patterns during development than in ageing, implying an increase in inter-individual heterogeneity [@Donertas2017]. 

### Research objectives

Whether age-related increase in heterogeneity is a universal phenomenon remains contentious. The studies analysed different organisms, organs, and age ranges, making the comparison difficult. Thus, we first employ different methodologies to assess the same dataset, to understand the extent of technical influence on these different results. Furthermore, where it can be detected, whether the increase in heterogeneity is a time-dependent process that starts at the beginning of life or whether this increase and its functional consequences are only seen after developmental processes are completed, have not yet been explored. In this study, we retrieved transcriptome data from three independent studies covering the whole lifespan, including data from diverse brain regions, and conducted a comprehensive analysis to identify the prevalence of age-related heterogeneity changes in human brain ageing compared with those observed during postnatal development. We further investigate the pathways and biological functions associated with the increased heterogeneity.

## Comparison of different methods to measure the change in gene expression heterogeneity with age {#veronikaresults}

In order to study the effect of different preprocessing methods and heterogeneity measures on gene expression heterogeneity during ageing, we used one of the biggest published human brain transcriptome datasets, generated using microarray technology [@Colantuoni2011]. We limited the age range to between 20 and 80 years, resulting in RNA expression data for 147 prefrontal cortex samples. We excluded prenatal, infant and childhood samples (up to 20 years old) because their expression levels are inherently coupled with developmental processes in the brain. 

### Methods to correct for batch effects

We applied two different batch correction strategies to account for technical and biological confounders:

1. Regression: Quantile normalisation followed by linear regression to correct for known co-variates, 
2. SVA: Quantile normalisation followed by Surrogate Variable Analysis 


We choose these two approaches as representatives of one supervised and one unsupervised approach. Linear regression requires the known confounders to be added as factors to the model, whereas SVA is unsupervised and deduces the factors from data. We also applied only quantile normalisation and ComBat to correct for batch effects and the results are available in the paper [@Kedlian2019]. For the brevity here I just summarise the results with these two methods. **Quantile Normalisation** was performed using the 'normalize.quantiles' function from the 'preprocessCore' R library. **Linear Regression** was applied considering: technical batches (N = 19), sex (N=2), race (N=4), post-mortem interval, RNA integrity number, and pH. **Surrogate Variable Analysis (SVA)** was applied using 20 inferred surrogate variables using the 'sva' package in R [@svapack]. Details of the methods are available in Section \@ref(veronikaMethods).

### Methods to measure age-related changes in gene expression heterogeneity

We also applied two different strategies to measure the change in the gene expression heterogeneity with age, namely *continuous* and *grouped* approaches. The continuous approach detects continuous monotonic change in variation from 20 to 80 years of age. The grouped approach compares the gene expression variation between two age groups: young (20 - 40 years old, N = 53) and old (60 - 80 years old, N = 22). 

In the *continuous approach*, we first fit a linear model to explain age-dependent change in expression, and then use the residuals from this model to represent the heterogeneity. To measure change in the expression heterogeneity with age, we calculate the Spearman correlation coefficient ($\Delta var (\rho)$) between the absolute value of residuals and age. 

In the *grouped approach*, we first generated a distribution of expected heterogeneity in gene expression for the young individuals and treated it as a null distribution to compare with the heterogeneity from the old individuals. We used interquartile range (IQR) as a measure of heterogeneity because it is robust to outliers. In order to calculate a distribution of expected heterogeneity in the young group, we randomly selected a subsample of 22 individuals (the number of samples in the old group) from the 53 individuals in the young group for 10,000 times and calculated IQR. The change in heterogeneity, $\Delta var(IQR)$, was measured as a fractional change in the IQR between old and young groups. The p-value was determined by calculating how many times we observed a value as extreme as $IQR_{old}$. 

Details of the methods are available in Section \@ref(veronikaMethods).

### Changes in heterogeneity at the transcriptome-level

The change in heterogeneity calculated using continuous approach, $\Delta var (\rho)$, ranged between -0.32 and 0.36 and were normally distriubted (Shapiro-Wilk test, p > 0.05) (Figure \@ref(fig:veronikaFig3)A). The distributions were significantly shifted towards positive values for both correction methods (Wilcoxon test, $p<2.2e^{-16}$, median values range between 0.01 and 0.03). Although the shift in the distribution was small, 57 and 63% of the genes showed increase in heterogeneity with age, for SVA and Regression corrections, respectively. However, we noted that the changes in heterogeneity calculated for each gene, using regression- and SVA-corrected data were only weakly correlated ($\rho = 0.35$, Figure \@ref(fig:veronikaFig3)B).

```{r veronikaFig3,out.width = '100%', fig.cap=c("Distributions of the $\\Delta var$-measures for all the genes (n = 16675) obtained in the continuous (A) and grouped (C) approaches. Increase in the heterogeneity with age, $\\Delta var > 0$, is colored in orange, while decrease in heterogeneity, $\\Delta var < 0$, is marked in blue. The red dashed  line depicts median of the distribution. The $\\Delta var(\\rho)$ distributions are normal with their mean and median values equal to  0.03  and  0.02  for  regression  and  SVA,  respectively;  The  $\\Delta var(IQR)$    distributions  are  moderately  skewed:  skewness  values  are  0.66  and  0.68  for  regression  and  SVA,  respectively.  The  mean  and  median  values  of  the  $\\Delta var(IQR)$  distribution  are  0.05  and  0.02  for  regression and 0.04 and 0.01 for SVA, respectively. Hexagonal heat maps illustrate relationship between regression and SVA-corrected measures  of  the  heterogeneity  for  each  gene,  obtained  in  continuous  - $\\Delta var(\\rho)$  (B)  and  grouped  - $\\Delta var(IQR)$  (D)  approaches.  The  color  gradient represents the density of the data. The linear regression line and the Spearman correlation estimate, $\\rho$, for the corresponding variables are shown on each graph."), fig.scap="Transcriptome-wide change in gene expression heterogeneity in ageing, using SVA and linear regression combined with continuous and grouped approaches."}
knitr::include_graphics(path = "figures/heterogeneity/veronikaFig3.png")
```

The distributions of the change in heterogeneity calculated using grouped approach, $\Delta var(IQR)$, were moderately skewed to the right and ranged from -0.7 to 2.1 for the regression corrected data and from -0.78 to 1.71 for the SVA corrected data (Figure \@ref(fig:veronikaFig3)C). The skew to the right was expected given that we calculate heterogeneity change as a fraction and thus, it was more sensitive to increase in heterogeneity. In both cases, the distributions demonstrated a significant deviation from zero (Wilcoxon test, $p<2.2e^{-16}$). 6% and 2% more genes showed higher heterogeneity in the old group, for regression- and SVA-corrected data, respectively. Similar to the continuous approach, the effect sizes calculated using different batch correction methods weakly correlated ($\rho = 0.24$, Figure \@ref(fig:veronikaFig3)D).

### Changes in heterogeneity at the gene-level

We then asked if we could detect any genes with a significant change in heterogeneity. Using the continuous approach, we did not detect any significant change in heterogeneity with age after multiple testing correction. The grouped approach leads to 741 and 746 differentially variable (DV) genes (FDR corrected p ≤ 0.05) using the regression and SVA correction, respectively. However, the two sets of DV genes had only 83 genes in common, one of which shows an opposite direction of change in the two sets. The correlation between $\Delta var(IQR)$ for regression and SVA corrected data is weak ($\rho$ = 0.24), but correlation increases when we select only the common DV genes ($\rho$ = 0.44). In agreement with our overview analysis above, we find twice as many DV genes with an increase in heterogeneity as those that decrease heterogeneity, using both correction methods: i) 533 genes increase and 208 decrease their heterogeneity after the regression correction, ii) 505 genes increase and 241 decrease their heterogeneity after the SVA correction. 

### Changes in heterogeneity at the pathway-level

Following the individual gene analysis, we explored whether genes that tend to increase or decrease heterogeneity with age are localised in particular functional groups. We performed multiple gene set enrichment analyses (GSEA) using the change in the heterogeneity with age ($\Delta var$) measures obtained in the continuous and grouped approaches on the gene sets from KEGG and Biological Process GO categories. We observed no genome-level significant enrichment in particular functional groups on the data either from the continuous (SVA correction), or the grouped approach (Regression and SVA corrections). However, we found that 4 pathways, namely beta-Alanine metabolism, Ras signaling pathway, Phosphatidylinositol signaling system, Bacterial invasion of epithelial cells (FDR corrected p ≤ 0.05) were enriched among the genes showing more heterogeneity of expression in the continuous approach (Regression correction). These pathways had positive normalised enrichment scores (NES), *i.e.* enrichment for the genes that increase heterogeneity with age. Moreover, these pathways also had positive NES for other approaches, even though they were not significant.

```{r veronikaFig5,out.width = '100%', fig.cap=c("The distribution of the heterogeneity measures ($\\Delta var$) for the genes within each pathway is represented as a box, encapsulating part of the distribution between 1st and 3rd quantile, median of the box is colored in black. (A) represents heterogeneity measure distributions for the continuous, while (B) for the grouped approaches. Pathways are ordered by increasing median. Boxes are colored in orange if the corresponding pathways have median $\\Delta var > 0$, and in blue, if median $\\Delta var < 0$. Text label on the plot shows percentage of pathways with median $\\Delta var > 0$. Red dashed line marks $\\Delta var = 0$. The mean values for the median across all pathways are 0.033 for $\\Delta var(\\rho)_{regression}$, 0.021 for $\\Delta var(\\rho)_{SVA}$, 0.033 for $\\Delta var(IQR)_{regression}$, and 0.027 for $\\Delta var(IQR)_{SVA}$."), fig.scap="Distributions of the heterogeneity measures obtained using a combination of continuous and grouped approaches with regression and SVA-correction for the individual pathways in KEGG database."}
knitr::include_graphics(path = "figures/heterogeneity/veronikaFig5.png")
```


The gene set enrichment analysis shows if there are particular gene sets that include the genes with the highest increase or decrease. Failing to detect such functional categories, we asked how the heterogeneity measures for the genes were distributed in the different functional groups of genes. For each of 310 KEGG pathways, encompassing 5922 unique genes, we analyzed the distributions of  $\Delta var$ measures, focusing on the median value for the change in heterogeneity (Figure \@ref(fig:veronikaFig5)A, B). In line with the overall tendencies we observed (Figure \@ref(fig:veronikaFig3)A, C), the majority of pathways contained a larger number of genes that become more variable with age, irrespective of the approach or correction method used. Although the increase in heterogeneity is ubiquitous and is observed across the majority of the pathways (74-94%), the increase is small (the mean value for the shift in distributions range between 0.021 and 0.033) – in accordance with the small, but significant increase observed in the distribution for all genes. Since the pathways are not mutually exclusive, we checked if there are particular genes that are present in many different pathways and cause the shift. However, no significant correlation between the pathway membership of gene and its heterogeneity measure ($\Delta var$) was detected.

### Methods {#veronikaMethods}

#### Data processing steps
**Dataset Selection:** We utilised one of the largest age-series human brain expression datasets, featuring 269 prefrontal cortex samples from healthy individuals and spanning the whole lifespan from development (prenatal samples) through ageing (80 years) [@Colantuoni2011]. These data were collected using microarray technology from people of both sexes and 4 races, namely African American (AA), Caucasian (CAUC), Hispanic (HISP) and Asian (AS). In the current analysis, we excluded fetal, childhood and early adulthood samples before the age of 20, thus limiting our sample size to 147. This was to exclude developmental processes taking place in the brain until the end of early adulthood, which exhibit discontinuous expression changes between early adulthood and ageing [@Donertas2017]. Our main motivation was to study changes in gene expression variability during ageing, considering 20 years old as a starting point.

**Data Characterisation:** The pre-processed data (loess normalisation was applied on the background corrected log2 intensity ratios (sample/reference)[@Colantuoni2011]); sample and gene (probe set to Entrez gene mapping) annotations were obtained from the NCBI Gene Expression Omnibus (GEO) at accession number GSE30272. Samples were processed in 19 batches, had different quality measurements, namely pH and RNA integrity number (RIN), and differed in the time of collection after death (post-mortem interval (PMI)). Using a PCA, we found no sample outliers as judged by visual inspection of the first two principal components. However, the relationship analysis between the above-mentioned factors (*i.e.* batch, RIN, PMI and others) and age yielded significant correlations for sex, post-mortem interval and RNA integrity, pointing to potential confounders in the data. We further checked the overlap between significantly differentially variable genes in our analysis and previously reported genes that are affected by PMI and detected only a limited overlap.

**Probe set to gene summarisation:** If one probe-set was mapped to several genes, it was deleted to avoid duplication. Conversely, when one gene had several probe-set expression values, they were averaged to obtain a unique gene expression value. In total 16675 genes were measured on the array.
*Batch correction:* To compensate for technical variation between samples, quantile normalisation (QN) was performed using the 'normalize.quantiles' function from the 'preprocessCore' R library. To differentiate between the age effect and the effect of the unwanted technical and biological variability, we have applied different expression correction strategies: linear regression of the known covariates, unsupervised estimation of covariates using surrogate variable analysis (SVA) [@Leek2007]. As a result, we analyzed the same data two times, corrected using QN+regression and QN+SVA. Different corrections work by adjusting for the different covariates in the linear model that explains the gene expression, namely: i) QN – no covariates were added; ii) QN+regression – considering technical batches (N = 19), sex (N=2), race (N=4), post-mortem interval, RNA integrity number, pH; iii) QN + SVA – 20 surrogate variables (SV) were inferred from the expression data using the 'sva' function from 'SVA' R library [@svapack].

#### Differential variability

**The continuous approach:** First, a linear model to fit gene expression during ageing, using $age^{0.25}$ and potential confounders, was constructed. Next, the Spearman correlation was calculated between the absolute values of the residuals, $|\epsilon_i|$ from the linear model and age. Consequently, Spearman correlation estimates were used as a measure of the change in variability, referred as $\Delta var_i(\rho)$. p values for the Spearman correlation estimates were corrected for multiple testing using FDR. FDR adjusted p ≤ 0.05 was used as a threshold to define significantly DV genes.

$$\Delta var_i(\rho)=\rho(|\epsilon_i,age|)$$

**The grouped approach:** First, a corrected expression matrix was obtained by removing the effect of covariates (see data processing steps) from the data using the residuals from a linear regression model. The 'grouped approach' is a custom resampling-based test designed to compare gene expression variability between young (20 – 40 years old) and old (60-80 years old) groups using an interquartile range (IQR). IQR corresponds to the difference between the 75th and 25th percentiles of the distribution and is considered to be a robust measure of variability, meaning it is not susceptible to outliers and departure from normality in the data. In order to adjust for the unequal sample size of the young (N = 53) and old (N = 22) groups, we, first, calculated a null distribution of the IQR values for the young group by resampling it 10,000 times with the size of the old group. Next, we calculated significance as a percentage of samples where $IQR_{old}$ was more extreme than $IQR_{young}$ and corrected it for multiple testing using FDR correction, q ≤ 0.05. The 'grouped' measure of change in the variability, $\Delta var_i(IQR)$, for the gene i, corresponds to the difference between IQR value for the old, $IQR_{i,old}$ , and $\overline{IQR_{i,young}}$ (*i.e.* mean IQR value from the young distribution), which is then divided by the latter, see formula:

$$\Delta var_i(IQR) = \dfrac{IQR_{i,old} - \overline{IQR_{i,young}}}{\overline{IQR_{i,young}}}$$

#### Gene Set Enrichment Analysis for KEGG pathways
$\Delta var$ measures from the differential variability analyses were used to perform gene set enrichment analysis, GSEA [@Subramanian2005] using the 'clusterProfiler' R library [@Yu2012]. 315 KEGG pathways with the size of between 10 and 500 genes were considered as gene sets for the GSEA.

#### Heterogeneity Distributions in Pathways

KEGG pathway to gene mapping was obtained from 'KEGGREST' R library and pathways were pre-filtered to contain between 5 and 500 genes. As a result, 310 KEGG pathways that comprise 5922 unique genes were used for the subsequent analysis. The boxplots illustrated distributions of the $\Delta var$ measure for genes in each pathway. Pathways were sorted according to their median $\Delta var$ measure in ascending order. The percentage of pathways that have their median $\Delta var$ above zero was calculated.

#### Distribution tests
Distributions of the $\Delta var$ - measures for all the genes were tested for normality using the Shapiro-Wilk test in R ('Shapiro.test' function) on the multiple subsamples, consisting of 5000 measures. Skewness of the distributions was calculated using the 'fBasics' function from 'BasicStatistics' R library.

## Temporal landscape of the changes in gene expression heterogeneity during brain development and ageing {#ulasresults}

To investigate how heterogeneity in gene expression changes with age, we used 19 published microarray datasets from three independent studies. Datasets included 1,010 samples from 17 different brain regions of 298 individuals whose ages ranged from 0 to 98 years (Supplementary Table S1, Figure \@ref(fig:hetFigS1)). In order to analyze the age-related change in gene expression heterogeneity during ageing compared to the change in development, we divided datasets into two subsets as development (0 to 20 years of age, n = 441) and ageing (20 to 98 years of age, n = 569). We used the age of 20 to separate pre-adulthood and adulthood based on commonly used age intervals in earlier studies (see Section \@ref(hetDatasets)). For the analysis, we focused only on the genes for which we have a measurement across all datasets (n = 11,137).

### Age-related change in gene expression levels

To quantify age-related changes in gene expression, we used a linear model between gene expression levels and age (see Section \@ref(hetMethods), Figure \@ref(fig:hetFigS2)). We transformed the ages to the fourth root scale before fitting the model as it provides relatively uniform distribution of sample ages across the lifespan, but we also confirmed that different age scales yield quantitatively similar results (Figure \@ref(fig:hetFigS3)). We quantified expression change of each gene in ageing and development periods separately and considered regression coefficients from the linear model ($\beta$ values) as a measure of age-related expression change (Figure \@ref(fig:hetFigS4)).


```{r hetFig1,out.width = '100%', fig.cap=c("Age-related change in gene expression during postnatal development and ageing. (a) Spearman correlations among age-related expression changes ($\\beta$  values) across datasets. The color of the squares indicates if the correlation between the corresponding pair of datasets (across β values of 11,137 common genes) is positive (red) or negative (blue), while darker color specifies a stronger correlation. Diagonal values were removed in order to enhance visuality. Annotation rows and columns indicate data source, brain region and period of each dataset. Hierarchical clustering was performed for each period separately (color of the dendrogram indicates periods) to determine the order of datasets. (b) Principal component analysis (PCA) of age-related expression changes during ageing and development. The analysis was performed on age-related expression change values of 11,137 common genes among all 38 datasets. The values of the first principal component on the x-axis and second principal component on the y-axis were drawn, where the values in the parenthesis indicate the variation explained by the corresponding principal component. Median Euclidean pairwise distances among development and ageing datasets calculated using PC1 and PC2 were annotated on the figure. Different shapes show different data sources and colors show development (dark orange) and ageing (blue) (c) Number of significant (FDR corrected p < 0.05) gene expression changes in development (left panel) and ageing (right panel). The x-axis shows the number of genes in the log scale. The color of the bars shows the direction of change, decrease (steel gray), and increase (orange). The exact number of genes are also displayed on the plot."), fig.scap="Age-related change in gene expression during postnatal development and ageing."}
knitr::include_graphics(path = "figures/heterogeneity/Figure1.png")
```

We first analyzed similarity in age-related expression changes across datasets by calculating pairwise Spearman's correlation coefficients among the $\beta$ values (Figure \@ref(fig:hetFig1)a). Both development (median correlation coefficient = 0.56, permutation test p < 0.001, Figure \@ref(fig:hetFigS5)) and ageing datasets (median correlation coefficient = 0.43, permutation test p = 0.003, Figure \@ref(fig:hetFigS5)) showed moderate correlation with the datasets within the same period. Although the difference between dataset correlations within development and ageing datasets was not significant (permutation test p = 0.1, Figure \@ref(fig:hetFigS6)), weaker consistency during ageing may reflect the stochastic nature of ageing, causing increased heterogeneity between ageing datasets. 

The principal component analysis (PCA) of age-related expression changes ($\beta$) revealed distinct clusters of development and ageing datasets (Figure \@ref(fig:hetFig1)b). Moreover, ageing datasets were more dispersed than development datasets (median pairwise Euclidean distances between PC1 and PC2 were 77 for ageing and 21 for development), which may again reflect stochasticity in gene expression change during ageing and can indicate more heterogeneity among different brain regions or datasets during ageing than in development.

We next identified genes showing significant age-related expression change (FDR-corrected p < 0.05), for development and ageing datasets separately (Figure \@ref(fig:hetFig1)c). Development datasets showed more significant changes compared to ageing (permutation test p = 0.003, Figure \@ref(fig:hetFigS6)), which may again indicate higher expression variability among individuals during ageing. The direction of change in development was mostly positive (14 datasets with more positive and 5 with more negative), whereas in ageing datasets, we observed more genes with a decrease in expression level (13 datasets with more  genes decreasing expression and 5 with no significant change, and 1 with an equal number of positive and negative changes).

### Age-related change in gene expression heterogeneity

To assess age-related change in heterogeneity, we obtained the unexplained variance (residuals) from the linear models used to calculate the change in gene expression level. For each gene in each dataset, we separately calculated Spearman's correlation coefficient ($\rho$) between the absolute value of residuals and age, irrespective of whether the gene shows a significant change in expression (Figure \@ref(fig:hetFigS2)). We considered $\rho$ values as a measure of heterogeneity change, where positive values mean an increase in heterogeneity with age. We also repeated this approach using loess regression instead of a linear model between expression level and age, and found high correspondence between $\rho$ values based on linear and loess regression models (Figure \@ref(fig:hetFigS7)). Still, loess regression was more sensitive to the changes in sample sizes and parameters and we therefore continued downstream analyses with the $\rho$ estimates based on the residuals from the linear model. 

```{r hetFig2,out.width = '100%', fig.cap=c("Age-related change in gene expression heterogeneity during development and ageing. The procedures are similar to those in the previous figure, except, age-related heterogeneity changes ($\\rho$ values) were used instead of expression changes ($\\beta$ values). (a) Spearman correlations among age-related heterogeneity changes ($\\rho$  values) across datasets. (b) Principal component analysis (PCA) of heterogeneity change with age. (c) The number of genes showing significant heterogeneity change in ageing and development."), fig.scap="Age-related change in gene expression heterogeneity during postnatal development and ageing."}
knitr::include_graphics(path = "figures/heterogeneity/Figure2.png")
```

We next asked if datasets show similar $\rho$, *i.e.* age-related changes in heterogeneity, by calculating pairwise Spearman's correlation between pairs of datasets, across shared genes (Figure \@ref(fig:hetFig2)a). Unlike the correlations among expression level changes, $\rho$ values did not show a higher consistency during development. In fact, although the difference is not significant (permutation test p = 0.2, Figure \@ref(fig:hetFigS6)), the median value of the correlation coefficients was higher in ageing (median correlation coefficient = 0.21, permutation test p = 0.24, Figure \@ref(fig:hetFigS5)), than in development (median correlation coefficient = 0.11, permutation test p = 0.25, Figure \@ref(fig:hetFigS5)). 

A principal component analysis (PCA) showed that, like expression change, heterogeneity change with age can also differentiate ageing datasets from development (Figure \@ref(fig:hetFig2)b). Similar to the pairwise correlations (Figure \@ref(fig:hetFig2)a), ageing datasets clustered more closely than development datasets, though the effect is small compared to the changes in expression level (median pairwise Euclidean distances are 41 and 44 for ageing and development, respectively). Both observations imply more similar changes in heterogeneity during ageing.

Using the p-values from Spearman's correlation between age and the absolute value of residuals for each gene, we then investigated the genes showing a significant change in heterogeneity during ageing and development (FDR corrected p-value < 0.05). We found almost no significant change in heterogeneity during development, except for the Colantuoni2011 dataset, for which we have high statistical power due to its large sample size. In ageing datasets, on the other hand, we observed more genes with significant changes in heterogeneity (permutation test p = 0.06, Figure \@ref(fig:hetFigS6)) and the majority of the genes with significant changes in heterogeneity tended to increase in heterogeneity (Figure \@ref(fig:hetFig2)c). However, the genes showing a significant change did not overlap across ageing datasets (Figure \@ref(fig:hetFigS8)).

Nevertheless, our analyses indicated relatively more consistent heterogeneity changes among datasets in ageing compared to development, implying that heterogeneity change could be a characteristic linked to ageing.

### Consistent increase in heterogeneity during ageing

As our previous analyses suggested age-related changes in heterogeneity can differentiate development from ageing and show more similarity during ageing, we sought to characterise the genes displaying such changes. Since the significance of the changes is highly dependent on the sample size, instead of focusing on significant genes identified within individual datasets, we leveraged upon the availability of multiple datasets and focused on their shared trends, capturing weak but reproducible trends across multiple datasets. Consequently, we used the level of consistency in age-related heterogeneity change across datasets to sort genes. 

```{r hetFig3,out.width = '100%', fig.cap=c("(a) Boxplots, showing distributions of age-related heterogeneity changes ($\\rho$ values) of 11,1137 common genes for each dataset and period separately. The dotted red line (vertical line at x = 0) reflects no change in heterogeneity. The difference between median heterogeneity change in ageing and development is given as a bar plot on the right panel. Datasets are ordered by the differences in median heterogeneity changes in ageing and development. (b) The relationship between expression and heterogeneity change with age. Spearman correlation analysis was performed between age-related expression changes ($\\beta$ values) and age-related heterogeneity changes ($\\rho$ values) of 11,137 common genes, separately for each dataset. The dotted gray line at y = 0 reflects no correlation between expression and heterogeneity. (c) Expected and observed consistency in the heterogeneity change across datasets in development and ageing. There is a significant shift toward heterogeneity increase in ageing (permutation test p<10-7) (lower panel), while there is no significant consistency in either direction in development (upper panel). The expected distribution is constructed using a permutation scheme that accounts for the dependence among datasets and is more stringent than random permutations"), fig.scap="a) Distributions of age-related changes in gene expression heterogeneity across datasets, b) The relationship between the age-related change in gene expression level and heterogeneity, c) Consistency in the change in gene expression heterogeneity."}
knitr::include_graphics(path = "figures/heterogeneity/Figure3.png")
```

We first examined profiles of age-related heterogeneity change in ageing and development. Among ageing datasets 18/19 showed more increase than decrease in heterogeneity with age (median $\rho$ > 0, *i.e.* higher numbers of genes with increase), while the median heterogeneity change in one dataset was zero. In development, on the other hand, only 5/19 datasets showed more increase in heterogeneity, while the remaining 14/19 datasets showed more decrease with age (median $\rho$ < 0) (Figure \@ref(fig:hetFig3)a). The age-related change in heterogeneity during ageing was significantly higher than during development (permutation test p < 0.001, Figure \@ref(fig:hetFigS6)). We also checked if there is a relationship between changes in heterogeneity during development and during ageing (*e.g.* if those genes that decrease in heterogeneity tend to increase in heterogeneity during ageing) but did not find any significant trend (Figure \@ref(fig:hetFigS9)).

A potential explanation why we see different patterns of heterogeneity change with age in development and ageing could be the accompanying changes in the expression levels, as it is challengeing to remove dependence between the mean and variance. To address this possibility, we first calculated Spearman's correlation coefficient between the changes in heterogeneity ($\rho$ values) and expression ($\beta$ values), for each dataset. Overall, all datasets had values close to zero, suggesting the association is not strong. Moreover, we saw an opposing profile for development and ageing; while the change in heterogeneity and expression were positively correlated in development, they showed a negative correlation in ageing (Figure \@ref(fig:hetFig3)b). 

Having observed both a more consistent heterogeneity change among ageing datasets (Figure \@ref(fig:hetFig2)a-b) and an increased heterogeneity in the ageing datasets compared to development (Figure \@ref(fig:hetFig3)a), we asked which genes show consistent increase in heterogeneity across datasets in ageing and development. We therefore calculated the number of datasets with an increase in heterogeneity during development and ageing for each gene (Figure \@ref(fig:hetFig3)c, \@ref(fig:hetFigS10)c). To calculate significance and expected consistency, while controlling for dataset dependence, we performed 1,000 random permutations of individuals' ages and re-calculated the heterogeneity changes. In development, there was no significant consistency in heterogeneity change in either increase or decrease. During ageing, however, there was a significant signal of consistent heterogeneity increase, *i.e.* more genes showed consistent heterogeneity increase across ageing datasets than randomly expected (Figure \@ref(fig:hetFig3)c, lower panel). We identified 147 common genes with a significant increase in heterogeneity across all ageing datasets (permutation test p < 0.001, Supplementary Table S2), whereas only one gene (*GPR137B*) showed a consistent decrease in heterogeneity during ageing. Based on our permutations, we estimated that 84/147 genes could be expected to have consistent increase just by chance, suggesting only ~40% true positives. In development, in contrast, there was no significant consistency in heterogeneity change in either direction (increase or decrease). Nevertheless, comparing the consistency in ageing and development, there was an apparent shift towards a consistent increase in ageing – even if we cannot confidently report the genes that become significantly more heterogeneous with age across multiple datasets.

### Heterogeneity trajectories  

```{r hetFig4,out.width = '100%', fig.cap=c("Clusters of genes showing a consistent heterogeneity increase in ageing (n = 147). Clustering was performed based on patterns of the change in heterogeneity, using the k-means clustering method. The x- and y-axes show age and heterogeneity levels, respectively. Mean heterogeneity change for the genes in each cluster was drawn by spline curves. The colors and line-types of curves specify different brain regions and data sources, respectively."), fig.scap="Clusters of genes showing a consistent heterogeneity increase in ageing"}
knitr::include_graphics(path = "figures/heterogeneity/Figure4.png")
```

We next asked if there are specific patterns of heterogeneity change, *e.g.* increase only after a certain age. We used the genes with a consistent increase in heterogeneity with age during ageing (n = 147) to explore the trajectories of heterogeneity change (Figure \@ref(fig:hetFig4)). Genes grouped with k-means clustering revealed three main patterns of heterogeneity increase (Supplementary Table S2): i) genes in clusters 3 and 7 show noisy but a steady increase throughout ageing, ii) genes in clusters 4, 5 and 8 show increase in early ageing but a later slight decrease, revealing a reversal (up-down) pattern, and iii) the remaining genes in cluster 1, 2 and 6 increase in heterogeneity dramatically after the age of 60. We next asked if these genes have any consistent heterogeneity change pattern in development (Figure \@ref(fig:hetFigS11)), but most of the clusters showed no or only weak age-related changes. We also analyzed the accompanying changes in mean expression levels for these clusters. Except for cluster 1, which shows a decrease in expression level at around the age of 60 and then shows a dramatic increase, all clusters show a steady scaled mean expression level at around zero, *i.e.* different genes in a cluster show different patterns (Figure \@ref(fig:hetFigS12)). 

We further tested the genes showing dramatic heterogeneity increase after the age of 60 (clusters 1, 2 and 6) for association with Alzheimer's Disease, as the disease incidence increases after 60 as well [@Evans1989]; however, we found no evidence for such an association (Figure \@ref(fig:hetFigS13)).

### Functional analysis

To examine the functional associations of heterogeneity changes with age, we performed gene set enrichment analysis (GSEA) using KEGG pathways [@Kanehisa2019], Gene Ontology (GO) categories [@Ashburner2000; @TheGeneOntologyConsortium2019], Disease Ontology (DO) categories [@Kibbe2015], Reactome pathways [@Fabregat2018], transcription factor (TF) targets (TRANSFAC) [@Matys2003], and miRNA targets (MiRTarBase) [@Chou2016]. Specifically, we rank-ordered genes based on the number of datasets that show a consistent increase in heterogeneity and asked if the extremes of this distribution are associated with the gene sets that we analyzed. There was no significant enrichment for any of the functional categories and pathways for the consistent changes in development. The significantly enriched KEGG pathways for the genes that become consistently heterogeneous during ageing included multiple KEGG pathways known to be relevant for ageing, including the longevity regulating pathway, autophagy [@Rubinsztein2011], mTOR signaling [@Johnson2013] and FoxO signaling [@Martins2016] (Figure \@ref(fig:hetFig5)a). Among the pathways with a significant association (listed in Figure \@ref(fig:hetFig5)a), only protein digestion and absorption, primary immunodeficiency, linoleic acid metabolism, and fat digestion and absorption pathways had negative enrichment scores, meaning these pathways were significantly associated with the genes having the least number of datasets showing an increase. However, it is important to note that this does not mean these pathways have a decrease in heterogeneity as the distribution of consistent heterogeneity levels is skewed (Figure \@ref(fig:hetFig3)c, lower panel). We also calculated if the KEGG pathways that we identified are particularly enriched in any of the heterogeneity trajectories we identified. Although we lack the necessary power to test the associations statistically due to small number of genes, we saw that i) group 1, which showed a stable increase in heterogeneity, is associated more with the metabolic pathways and mRNA surveillance pathway, ii) group 2, which showed first an increase and a slight decrease at later ages, is associated with axon guidance, mTOR signaling, and phospholipase D signaling pathways, and iii) group 3, which showed a dramatic increase after age of 60, is associated with autophagy, longevity regulating pathway and FoxO signaling pathways. The full list is available as Figure \@ref(fig:hetFigS14).

```{r hetFig5,out.width = '100%', fig.cap=c("Functional analysis of consistent heterogeneity changes. (a) Distribution of consistent heterogeneity increase for the significantly enriched KEGG pathways, in development and ageing. x- and y-axes show the number of datasets with a consistent increase and the density for each significant pathway, respectively. The dashed red line shows x = 9.5, which is the middle point for 19 datasets, representing no tendency to increase or decrease. Values higher than 9.5, shown with red color, indicate an increase in heterogeneity while values lower than 9.5, shown with blue color, indicate a decrease in heterogeneity and the darkness shows the consistency in change across datasets. b) The longevity regulating pathway (KEGG Pathway ID: hsa04211), exemplifying the distribution of the genes (circles), their heterogeneity across datasets (color – the same color scheme as panel (a)), and their relationship in the pathway (edges)."), fig.scap="Functional analysis of consistent heterogeneity changes."}
knitr::include_graphics(path = "figures/heterogeneity/Figure5.png")
```

The distribution of consistent heterogeneity in development and ageing also showed a clear difference. The pathway scheme for the longevity regulating pathway (Figure \@ref(fig:hetFig5)b), colored based on the number of datasets with a consistent increase, shows how particular genes compare between development and ageing. Other significantly enriched gene sets, including GO, Reactome, TF and miRNA sets, are included as Supplementary Tables S3-10. In general, while the consistent heterogeneity changes in development did not show any enrichment (except for miRNAs, see Supplementary Table S10), we detected a significant enrichment for the genes that become more heterogeneous during ageing, with the exception that Disease Ontology terms were not significantly associated with consistent changes in either development or ageing. The gene sets included specific categories such as autophagy and synaptic functions as well as broad functional categories such as regulation of transcription and translation processes, cytoskeleton or histone modifications. We also performed GSEA for each dataset separately and confirmed that these pathways show consistent patterns in ageing
<!-- (Figure \@ref(fig:hetFigS15), \@ref(fig:hetFigS16), \@ref(fig:hetFigS17), \@ref(fig:hetFigS18), \@ref(fig:hetFigS19)) -->
. There were 30 significantly enriched transcription factors, including EGR and FOXO, and 99 miRNAs (see Supplementary Table S8-9 for the full list). We also asked if the genes that become more heterogenous consistently across datasets are known ageing-related genes, using the GenAge Human gene set [@Tacutu2018], but did not find a significant association (Figure \@ref(fig:hetFigS20)).

It has been reported that the total number of distinct regulators of a gene (apart from its specific regulators) is correlated with gene expression noise [@Barroso2018]. Accordingly, we asked if the total number of transcription factors (TFs) or miRNAs regulating a gene might be related to the heterogeneity change with age (Figure \@ref(fig:hetFig6)). We calculated the correlations between the total number of regulators and the heterogeneity changes and found a mostly positive (18 / 19 for miRNA and 15 / 19 for TFs), and higher correlation between change in heterogeneity and the number of regulators in ageing (p = 0.007 for miRNA and p = 0.045 TFs). We further tested the association while controlling for the expression changes in development and ageing since regulation of expression changes during development could confound the relationship. However, we found that the pattern is mainly associated with the genes that show a decrease in expression during ageing, irrespective of their expression during development (Figure \@ref(fig:hetFigS21)). 

```{r hetFig6,out.width = '80%', fig.cap=c("Correlation between the change in heterogeneity and number of transcriptional regulators, i.e. miRNA and transcription factors. Each point represents a dataset, and the color shows the data source. p-values are calculated using a permutation test. The dashed line at y = 0 shows zero correlation."), fig.scap="Correlation between the change in heterogeneity and number of transcriptional regulators."}
knitr::include_graphics(path = "figures/heterogeneity/Figure6.png")
```

We further tested if genes with a consistent heterogeneity increase in ageing are more central in the protein interaction network using STRING database [@VonMering2005]. Using multiple cutoffs and repeating the analysis, we observed a higher degree of interactions for the genes with increasing heterogeneity (Figure \@ref(fig:hetFigS22)). 

@Johnson2015 previously compiled a list of traits that are age-related and have been sufficiently tested for genome-wide associations (n = 39). Using the genetic associations for GWAS Catalog traits, we tested if there are significantly enriched traits for the consistent changes in heterogeneity during ageing (Supplementary Table S11). Although there was no significant enrichment, all these age-related terms had positive enrichment scores, *i.e.* they all tended to include genes that consistently become more heterogeneous with age during ageing.

Using cell-type specific transcriptome data generated from FACS-sorted cells in mouse brain [@Cahoy2008], we also analyzed if there is an association between genes that become heterogeneous with age and cell-type specific genes, which could be expected if brain cell-type composition progressively varied among individuals with age. Although there was an overlap with oligodendrocytes and myelinated oligodendrocytes, there was no significant enrichment (which could be attributed to low power due to small overlap – n= 9 genes - between ageing and cell-type specific expression datasets) (Figure \@ref(fig:hetFigS23)). We further analyzed if the cell-type proportions change or become heterogeneous with age, using a deconvolution method. Overall, we see an increase in myelinated oligodendrocytes in development and astrocytes in ageing (Figure \@ref(fig:hetFigS24)a). Focusing specifically on the 147 genes that become consistently heterogeneous with age, we do not see any consistent pattern in ageing (Figure \@ref(fig:hetFigS24)b), suggesting these genes are not more relevant for certain cell-types. Moreover, we checked if the cell-type proportions, calculated for all genes and only for the 147 genes, become heterogeneous with age during ageing, and found that all cell types seem to become heterogeneous when the heterogeneous gene set analyzed, again suggesting their effects are more likely to be broad and not cell-type specific (Figure \@ref(fig:hetFigS24)d). 

### Methods {#hetMethods}

#### Dataset collection {#hetDatasets}

In this study, we performed re-analysis of publicly available transcriptome datasets to test age-related change in gene expression heterogeneity. All data collection in these previous studies were performed in accordance with relevant guidelines, regulations and approved experimental protocols, including informed consents for the use of samples for research from all donors or their next of kin.

**Microarray datasets:** Raw data used in this study were retrieved from the NCBI Gene Expression Omnibus (GEO) from three different sources (Supplementary Table S1). All three datasets consist of human brain gene expression data generated on microarray platforms. In total, we obtained 1017 samples from 298 individuals, spanning the whole lifespan with ages ranging from 0 to 98 years (Figure \@ref(fig:hetFigS1)).

**RNA sequencing dataset:** We used the transcriptome data generated by the GTEx Consortium (v6p) [@Ardlie2015]. We only used the samples with a death circumstance of 1 (violent and fast deaths due to an accident) and 2 (fast death of natural causes) on the Hardy Scale excluding individuals who died of illnesses. As we focus only on the brain, we used all 13 brain tissue data in GTEx. We thus analyzed 623 samples obtained from 99 individuals.

**Separating datasets into development and ageing datasets:** To differentiate changes in gene expression heterogeneity during ageing from those during development, we used the age of 20 to separate pre-adulthood from adulthood. It was shown that the age of 20 corresponds to the average first age of reproduction in human societies [@Walker2006]. Structural changes after the age of 20 in the human brain were previously linked to age-related phenotypes, specifically neuronal shrinkage and a decline in total length of myelinated fibers [@Sowell2004]. Earlier studies examining age-related gene expression changes in different brain regions also showed a global change in gene expression patterns after the age of 20 [@Somel2010; @Donertas2017; @Colantuoni2011]. Thus, consistent with these studies, we separated datasets using the age of 20 into development (0 to 20 years of age, n = 441) and ageing (20 to 98 years of age, n = 569).

#### Preprocessing

**Microarray datasets:** RMA correction (using the 'oligo' library in R) [@Carvalho2010] and log2 transformation were applied to Somel2011 and Kang2011 datasets. For the Colantuoni2011 dataset, as there was no public R package to analyze the raw data, we used the preprocessed data deposited in GEO, which had been loess normalised by the authors. We quantile normalised all datasets using the 'preprocessCore' library in R [@Bolstad2019].  Notably, our analysis focused on consistent patterns across datasets, instead of considering significant changes within individual datasets. Since we don't expect random confounding factors to be shared among datasets, we used quantile normalisation to minimise the effects of confounders, and we treated consistent results as potentially a biological signal. We also applied an additional correction procedure for Somel2011 datasets, in which there was a batch effect influencing the expression levels, as follows: for each probeset (1) calculate mean expression (M), (2) scale each batch separately (to mean = 0, standard deviation = 1), (3) add M to each value. We excluded outliers given in Supplementary Table S1, through a visual inspection of the first two principal components for the probeset expression levels as in @Donertas2017 and @Donertas2018. We mapped probeset IDs to Ensembl gene IDs 1) using the Ensembl database, through the 'biomaRt' library [@Durinck2009] in R for the Somel2011 dataset, 2) using the GPL file deposited in GEO for Kang2011, as probeset IDs for this dataset were not complete in Ensembl, and 3) using the Entrez gene IDs in the GPL file deposited in GEO for the Colantuoni2011 dataset and converting them into Ensembl gene IDs using the Ensembl database, through the 'biomaRt' library in R. Lastly, we scaled expression levels for genes (to mean = 0, standard deviation = 1) using the 'scale' function in R. Age values of individuals in each dataset were converted to the fourth root of age (in days) to have a linear relationship between age and expression both in development and ageing. However, we repeated the analysis using different age scales and confirmed that the results were quantitatively similar (Figure \@ref(fig:hetFigS3)).

**RNA sequencing dataset:** The genes with median RPKM value of 0 were excluded from the dataset. The RPKM values provided in the GTEx data were log2 transformed and quantile-normalised. Similar to the microarray data, we excluded the outliers based on the visual inspection of the first and second principal components (Supplementary Table S1). In GTEx, ages are given as 10 year intervals. We therefore used the middle point of these age intervals to represent that individual's age.

#### Age-related expression change

We used linear regression to assess the relationship between age and gene expression. The model used in the analysis is:

\begin{equation}
    Y_i = \beta_{i_0} + \beta_{i_1}*Age^{1/4} + \epsilon_i
  (\#eq:heteq1)
\end{equation}

where $Y_i$ is the scaled log2 expression level for the $i^{th}$ gene, $\beta_{i_0}$ is the intercept, $\beta_{i_1}$ is the slope, and $\epsilon_i$ is the residual. We performed the analysis for each dataset (development and ageing datasets separately) and considered the $\beta_1$ value as a measure of change in expression. p-values obtained from the model were corrected for multiple testing according to Benjamini and Hochberg procedure [@Benjamini1995] by using 'p.adjust' function in R.

#### Age-related heterogeneity change

In order to quantify the age-related change in gene expression heterogeneity, we calculated Spearman's correlation coefficient ($\rho$). The correlations were calculated between the absolute values of residuals obtained from equation \@ref(eq:heteq1) and the fourth root of individual age. We regarded the absolute values of residuals as a measure of heterogeneity. Therefore, high positive correlation coefficients suggest that heterogeneity increases with age, whereas strong negative correlation implies heterogeneity decreases with age. p-values were calculated from the correlation analysis and corrected for multiple testing with the Benjamini and Hochberg method using the 'p.adjust' function in R.
We further confirmed our results using a different measure of heterogeneity, Breusch-Pagan test, implemented as 'ncvTest' function in the 'car' package in R [@Fox2019] (Figure \@ref(fig:hetFigS32)).


#### Principal Component Analysis

We conducted principal component analysis on both age-related changes in expression ($\beta$) and heterogeneity ($\rho$). We followed a similar procedure for both analyses, in which we used the 'prcomp' function in R.  The analysis was performed on a matrix containing $\beta$ values (for the change in expression level) and $\rho$ values (for the change in heterogeneity), for 11,137 commonly expressed genes for all 38 development and ageing datasets. In each dataset, the estimates of expression change ($\beta$) or heterogeneity change ($\rho$) values were scaled for each dataset before calculating principal components.

#### Permutation test

We used a permutation scheme that we developed earlier [@Donertas2017; @Donertas2018], taking into account that samples across Somel2011 and Kang2011 datasets are not independent (*i.e.* these datasets include multiple samples from the same individuals for different brain regions). Specifically, we first randomly permuted ages among individuals, not samples, for 1,000 times in each data source, using the 'sample' function in R. Next, we assigned ages of individuals to their corresponding samples, making sure that multiple samples from the same individual annotated with the same age, retaining the dependency between datasets. Then, we calculated age-related expression and heterogeneity change for each dataset, using permuted ages. For the tests related to the changes in gene expression with age, we used a linear model between gene expression levels and the randomised ages. In contrast, for the tests related to the changes in heterogeneity with age, we measured the correlation between the randomised ages and the absolute value of residuals from the linear model that we obtained from equation \@ref(eq:heteq1) using non-randomised ages for each gene. In this way, we preserved the relationship between age and expression, and we were able to ensure that our regression model was appropriate for calculating age-related heterogeneity change [@Somel2006]. Using expression and heterogeneity change estimates calculated using permuted ages, we tested (a) if the correlation of expression (and heterogeneity) change in ageing is higher than in development datasets; (b) if the correlations of expression (and heterogeneity) change in development and in ageing datasets are significantly higher than null expectation; (c) if the number of genes showing significant change in expression (and heterogeneity) is significantly higher in ageing than in development datasets; (d) if the overall increase in age-related heterogeneity during ageing is significantly higher than development; (e) if the observed consistency in heterogeneity increase is significantly different from expected. All tests using permuted ages were performed one-tailed. We also demonstrate that our permutation strategy is more stringent than random permutations in Figure \@ref(fig:hetFigS10), giving the distributions calculated using both dependent permutations and random permutations.

To test the overall correlation within development or ageing datasets for the changes in expression ($\beta$) and heterogeneity ($\rho$), we calculated median correlations among independent three subsets of datasets (one Kang2011, one Somel2011 and the Colantuoni2011 dataset), taking the median value calculated for each possible combination of independent subsets (16 x 2 x 1 = 32 combinations). Using 1,000 permutations of individuals' ages, we generated an expected distribution for the median correlation coefficient for triples and compared these with the observed values, asking how many times we observe a higher value. We used this approach to calculate expected median correlation among development (and ageing) datasets, because the number of independent pairwise comparisons are outnumbered by the number of dependent pairwise comparisons, causing low statistical power.

To further test the significance of the difference between correlations among development and ageing datasets, we calculated the median difference in correlations between ageing and development datasets for each permutation. We next constructed the null distribution of 1,000 median differences and calculated empirical p-values comparing the observed differences with these null distributions. Next, to test the significance of the difference in the number of significantly changing genes between development and ageing, we calculated the difference in the number of genes showing significant change between development and ageing datasets for each permutation. Empirical p-values were computed according to observed differences. Likewise, to test if the overall increase in age-related heterogeneity during ageing is significant compared to development, we computed median differences between median heterogeneity change values of each ageing and development dataset, for each permutation, followed by an empirical p-value calculation to answer if the ageing datasets have a higher increase in age-related heterogeneity.

#### Expected heterogeneity consistency

Expected consistency in heterogeneity change was calculated from heterogeneity change values ($\rho$) measured using permuted ages. For each permutation, we first calculated the total number of genes showing consistent heterogeneity increase for N number of datasets (N = 0, ..., 19). To test if observed consistency significantly differed from the expected, we compared observed consistency values to the distribution of expected numbers, by performing a one-sided test for the consistency in N number of datasets, N = 1, …, 19.

#### Clustering

We used the k-means algorithm ('kmeans' function in R) to cluster genes showing consistent heterogeneity change (n=147) according to their heterogeneity profiles. We first took the subset of the heterogeneity levels (absolute value of the residuals from equation \@ref(eq:heteq1)) to include only the genes that show a consistent increase with age and then scaled the heterogeneity levels to the same mean and standard deviation. Since the number of samples in each dataset is different, just running k-means on the combined dataset would not equally represent all datasets. Thus, we first calculated the spline curves for scaled heterogeneity levels for each gene in each dataset (using the 'smooth.spline' function in R, with three degrees of freedom). We interpolated at 11 (the smallest sample size) equally distant age points within each dataset. Then we used the combined interpolated values to run the k-means algorithm with k = 8, a liberal choice, given the total number of genes being 147.

To test association of the clusters with Alzheimer's Disease, we retrieved overall AD association scores of the 147 consistent genes (n = 40) from the Open Targets Platform [@Carvalho-Silva2019].

#### Functional Analysis

We used the "clusterProfiler" package in R to run Gene Set Enrichment Analysis (GSEA), using Gene Ontology (GO) Biological Process (BP), GO Molecular Function (MF), GO Cellular Compartment (CC), Reactome, Disease Ontology (DO), and KEGG Pathways. We used Normalised Enrichment Scores (NES) from ClusterProfiler package, which gives a weighted Kolmogorov-Smirnov (KS) test statistic divided by the mean of KS statistics obtained from random permutations. We performed GSEA on all gene sets with a size between 5 and 500, and we corrected the resulting p-values with the Benjamini and Hochberg correction method. To test if the genes with a consistent increase or decrease in their expression are associated with specific functions, we used the number of datasets with a consistent increase to run GSEA. Since we are running GSEA using number of datasets showing consistency, our data includes many ties, potentially making the ranking ambiguous and non-robust. In order to assess how robust our results are, we ran GSEA 1,000 times on the same data and counted how many times we observed the same set of KEGG pathways as significant (Supplementary Table S3). The lowest number among the pathways with a significant positive enrichment score was 962 out of 1,000 (Phospholipase D signaling pathway). Moreover, we repeated the same analysis using the heterogeneity change levels ($\rho$), instead of using the number of datasets with a consistent change, for each dataset to confirm the gene sets are indeed associated with the increase or decrease in heterogeneity
<!-- (Figures \@ref(fig:hetFigS15),\@ref(fig:hetFigS16),\@ref(fig:hetFigS17),\@ref(fig:hetFigS18),\@ref(fig:hetFigS19)) -->
. We visualised the KEGG pathways using 'KEGGgraph' library in R and colored the genes by the number of datasets that show an increase.

We also performed an enrichment analysis of the transcription factors and miRNA to test if specific TFs or miRNAs regulate the genes that become more heterogeneous consistently. We collected gene-regulator association information using the Harmonizome database [@Rouillard2016], 'MiRTarBase microRNA Targets' (12086 genes, 596 miRNAs) and 'TRANSFAC Curated Transcription Factor Targets' (13216 genes, 201 TFs) sets. We used the 'fgsea' package in R, which allows GSEA on a custom gene set. We tested the association for each regulator with at least 10 and at most 500 targets. Moreover, we tested if the number of regulators is associated with the change in heterogeneity. We first calculated the correlation between heterogeneity change with age (or the number of datasets with an increase in expression heterogeneity) and the number of TFs or miRNAs regulating that gene, for ageing and development separately. We repeated the analysis while accounting for the direction of expression changes in these periods (*i.e.* separating genes into down-down, down-up, up-down, and up-up categories based on their expression in development and ageing, Supplementary Figure \@ref(fig:hetFigS21)). To test the difference in the correlations between ageing and development, we used 1,000 random permutations of the number of TFs. For each permutation, we randomised the number of TFs and calculated the correlation between heterogeneity change (or the number of datasets with an increase in heterogeneity) and the randomised numbers. We then calculated the percentage of datasets where ageing has a higher correlation than development. Using the distribution of percentages, we tested if the observed value is expected by chance.

#### Protein-protein interaction network analysis

We downloaded all human protein interaction data from the STRING database (v11) [@VonMering2005]. Ensembl Peptide IDs are mapped to Ensembl Gene IDs using the 'biomaRt' package in R. Here we aimed to test whether genes showing consistent increase in heterogeneity have a different number of interactors than other genes. For this we calculated the degree distributions for the genes that become consistently more heterogeneous with age and all remaining genes using different cutoffs for interaction confidence scores. In order to calculate the significance of the difference, we i) calculated the number of interactors (degree) for each gene, ii) for 10,000 times, randomly sampled k genes from all interactome data (k = number of genes that become heterogeneous with age across all datasets and have interaction information in STRING database, after filtering for a cutoff), iii) calculated the median of degree for each sample. We then calculated an empirical p-value by asking how many of these 10,000 samples we see a median degree that is equivalent to or higher than our original value. The number of genes and interactions after each cutoff are given in Figure \@ref(fig:hetFigS22).

#### Cell-type specificity analysis

Using FACS-sorted cell-type specific transcriptome data from the mouse brain [@Cahoy2008], we checked if there is any overlap between genes that become heterogeneous with age and cell-type specific genes. We downloaded the dataset from the GEO database (GSE9566) and preprocessed it by performing: i) RMA correction using the 'affy' package in R [@Gautier2004], ii) log2 transformation, iii) quantile normalisation using the 'preprocessCore' package in R [@Bolstad2019], iv) mapping probeset IDs to first mouse genes, and then human genes. We only included genes that have one to one orthologs in humans, after filtering out probesets that map to multiple genes.

We first defined the cell-type specific genes by calculating an effect size (Cohen's D) for each gene and cell type and identifying genes that have an effect size higher than or equal to 2 as specific to that cell type. At this cutoff, there was no overlap between cell type specific gene lists. To test for association between heterogeneity and cell type specificity, we used the Fisher's exact test using the R 'fisher.test' function.

Following another approach, we analyzed the relative cell-type contribution to the expression profiles and calculated both how the level and heterogeneity of the relative contributions change with age. This analysis requires positive expression values and since the Colantuoni2011 dataset is loess-normalised, we only analyzed Kang2011 and Somel2011 datasets. To determine the relative contribution of different cell-types we used a linear regression based approach, where the contributions are determined based on the following formula:

\begin{equation}
    Expr = \alpha + \beta_{A}\zeta_{A} + \beta_{Oli}\zeta_{Oli} + \beta_{MOli}\zeta_{MOli} + \beta_{OPC}\zeta_{OPC} + \beta_{N}\zeta_{N} + \epsilon
  (\#eq:heteq2)
\end{equation}

Where; A: Astrocytes, Oli: Oligodendrocytes, M_Oli: Myelinated Oligodendrocytes, OPC: Oligodendrocyte Precursor Cells, N: Neurons. $\beta$'s represent the regression coefficients estimate the relative contributions of each cell type, $\zeta$'s represent expression level of each cell type (averaged across replicates), and $\epsilon$ represents residuals.

#### Effect of sex-specific gene expression on heterogeneity

To test the effect of sex on increased heterogeneity during ageing, we first obtained residuals from equation \@ref(eq:heteq1) for 147 consistent genes. Then, we performed two sample Wilcoxon test, using 'wilcox.test' function in R, to test if the distribution of residuals differs significantly between sexes. As we used the real values of residuals here (not absolute values), we would expect a significant difference in their distribution if the expression change is in different directions between sexes. p-values obtained from Wilcoxon test for each gene in each dataset separately were corrected for multiple testing with the Benjamini and Hochberg method using the 'p.adjust' function in R. The number of genes showing a significant difference, and their consistency among ageing datasets are shown in Figure \@ref(fig:hetFigS28).

#### Code Availability
All analysis was performed using R and the code to calculate heterogeneity changes with age is available as an R package 'hetAge', documented at https://mdonertas.github.io/hetAge/. 'ggplot2' [@Wickham2017], 'ggpubr' [@Kassambara2018], and 'pheatmap' [@Kolde2019], R libraries were used for the visualisation.


## Discussion

Ageing is characterised by a gradual decrease in the ability to maintain homeostatic processes, which leads to functional decline, age-related diseases, and eventually to death. This age-related deterioration, however, is thought as not a result of expression changes in a few individual genes, but rather as a consequence of an age-related alteration of the whole genome, which could be a result of an accumulation of both epigenetic and genetic errors in a stochastic manner [@Enge2017; @Vijg2004]. This stochastic nature of ageing impedes the identification of conserved age-related changes in gene expression from a single dataset with a limited number of samples.

In this study, we examined 19 gene expression datasets compiled from three independent studies to identify the changes in gene expression heterogeneity with age. While all datasets have samples representing the whole lifespan, we separated postnatal development (0 to 20 years of age) and ageing (20 to 98 years of age) by the age of 20, as this age is considered to be a turning-point in gene expression trajectories [@Donertas2017]. We implemented a regression-based method and identified genes showing a consistent change in heterogeneity with age, during development and ageing separately. At the single gene level, we did not observe significant age-related heterogeneity change in most of the datasets, possibly due to insufficient statistical power due to small sample sizes and the subtle nature of the phenomenon. We hence took advantage of a meta-analysis approach and focused on consistent signals among datasets, irrespective of their effect sizes and significance. Although this approach fails to capture patterns that are specific to individual brain regions, it identifies genes that would otherwise not pass the significance threshold due to insufficient power. Furthermore, we demonstrated that our method is robust to noise and confounding effects within individual datasets.  

By analyzing age-related gene expression changes, we first observed that there are more significant and more similar changes during development than ageing. Additionally, genes showing significant change during ageing tended to decrease in expression (Figure \@ref(fig:hetFig1)). These results can be explained by the accumulation of stochastic detrimental effects during ageing, leading to a decrease in expression levels [@Lu2004]. Our initial analysis of gene expression changes suggested a higher heterogeneity between ageing datasets.

We next focused on age-related heterogeneity change between individuals and found a significant increase in age-related heterogeneity during ageing, compared to development. Notably, increased heterogeneity is not limited to individual brain regions, but a consistent pattern across different regions during ageing.  We found that age-related heterogeneity change is more consistent among ageing datasets, which may reflect an underlying systemic mechanism. Further, a larger number of genes showed more significant heterogeneity changes during ageing than in development, and the majority of these genes tended to have more heterogeneous expression. 

It was previously proposed that somatic mutation accumulations [@Lu2004; @Vijg2004; @Lodato2018; @Lombard2005] and epigenetic regulations [@Cheung2018] might be associated with transcriptome instability. While @Enge2017 and @Lodato2018 suggested that genome-wide substitutions in single cells are not so common as to influence genome stability and cause transcriptional heterogeneity at the cellular level, epigenetic mechanisms may be relevant. Although we cannot test age-related somatic mutation accumulation and epigenetic regulation in this study, an alternative mechanism might be related to transcriptional regulation, which is considered to be inherently stochastic [@Maheshri2007]. Several studies demonstrated that variation in gene expression is positively correlated with the number of TFs controlling gene's regulation [@Barroso2018]. We also found that genes with a higher number of regulators and a decrease in expression during ageing become more heterogeneous. Further, significantly enriched TFs include early growth response (EGF), known to be regulating the expression of many genes involved in synaptic homeostasis and plasticity, and FOXO TFs, which regulate stress resistance, metabolism, cell cycle arrest and apoptosis. Together with these studies, our results support that transcriptional regulation may be associated with age-related heterogeneity increase during ageing and may have important functional consequences in brain ageing.

We next confirmed that observed increase in heterogeneity was not a result of low statistical power (Figure \@ref(fig:hetFigS1)) or a technical artifact (Figure \@ref(fig:hetFig3)b, \@ref(fig:hetFigS24), \@ref(fig:hetFigS25)). Specifically, we tested whether increased heterogeneity during ageing can be a result of the mean-variance relationship, but we found no significant effect that can confound our results. In fact, the mean-variance relationship in development and ageing showed opposing profiles. We further analyzed this by grouping genes based on their expression in development and ageing (Figure \@ref(fig:hetFigS24)). The genes that decrease in expression both in development and ageing showed the most opposing profiles in terms of the mean-variance relationship, which could suggest that the decrease in development are more coordinated and well-regulated whereas the decrease in ageing occurs due to stochastic errors. Another potential confounder is the post-mortem interval (PMI), which is the time between death and sample collection. Since we do not have this data for all datasets we analyzed, we could not account for PMI in our model. However, using the list of genes previously suggested as associated with PMI [@Zhu2017], we checked if the consistency among ageing datasets could be driven by PMI. Only 2 PMI-associated genes were among the 147 that become consistently heterogeneous, and the distribution also suggested there is no significant relationship (Figure \@ref(fig:hetFigS25)). We also confirmed that the increase in heterogeneity is not caused by outlier samples in datasets (Figure \@ref(fig:hetFigS26)) or by the confound of sex with age (Figure \@ref(fig:hetFigS27)). Moreover, we asked if differential expression trajectories between different sexes may contribute to the observed heterogeneity. Specifically, we tested if the distribution of residuals differs between males and females using 147 consistent genes. We observed that only 15 of 147 genes show a significant difference in at least one dataset (Figure \@ref(fig:hetFigS28)). 

Gene set enrichment analysis of the genes with increased heterogeneity with age revealed a set of significantly enriched pathways that are known to influence ageing, including longevity regulating pathway, autophagy, mTOR signaling pathway (Figure \@ref(fig:hetFig5)a). Furthermore, GO terms shared among these genes include some previously identified common pathways in ageing and age-related diseases
<!-- (Figures \@ref(fig:hetFigS16), \@ref(fig:hetFigS17), \@ref(fig:hetFigS18)) -->
. We have also tested if these genes are associated with age-related diseases through GWAS, and although not significant, we found a positive association with all age-related traits defined in @Johnson2015. Overall, these results indicate the effect of heterogeneity on pathways that modulate ageing and may reflect the significance of increased heterogeneity in ageing. Importantly, we identified genes that are enriched in terms related to neuronal and synaptic functions, such as axon guidance, neuron to neuron synapse, postsynaptic specialisation, which may reflect the role of increased heterogeneity in synaptic dysfunction observed in the mammalian brain, which is considered to be a major factor in age-related cognitive decline [@Morrison2012]. We also observed genes that become more heterogeneous with age consistently across datasets are more central (*i.e.* have a higher number of interactions) in a protein-protein interaction network (Figure \@ref(fig:hetFigS22)). Although this could mean the effect of heterogeneity could be even more critical because it affects hub genes, another explanation is research bias that these genes are studied more than others.

### Limitations

One important limitation of our study is that we analyze microarray-based data. Since gene expression levels measured by microarray do not reflect an absolute abundance of mRNAs, but rather are relative expression levels, we were only able to examine relative changes in gene expression. A recent study analyzing single-cell RNA sequencing data from the ageing Drosophila brain identified an age-related decline in total mRNA abundance [@Davie2018]. It is also suggested that, in microarray studies, genes with lower expression levels tend to have higher variance [@Aris2004]. In this context, whether the change in heterogeneity is a result of the total mRNA decay is an important question. As an attempt to see if the age-related increase in heterogeneity is dependent on the technology used to generate data, we repeated the initial analysis using RNA sequencing data for the human brain, generated by GTEx Consortium [@Ardlie2015] (Figures \@ref(fig:hetFigS29), \@ref(fig:hetFigS30), \@ref(fig:hetFigS31)). Nine out of thirteen datasets displayed more increase than decrease in heterogeneity during ageing, consistent with 18/19 microarray datasets, while the remaining four datasets showed the opposite pattern (BA24, cerebellar hemisphere, cerebellum and substantia nigra). Unlike what we observed for the microarray datasets, the change in expression levels and heterogeneity were strongly positively correlated (Figure \@ref(fig:hetFigS31)). Unfortunately, average expression levels and variation levels in RNA sequencing is challenging to disentangle. Thus, the biological relevance of the relationship between the age-related change in expression levels and expression heterogeneity still awaits to be studied through novel experimental and computational approaches. Nevertheless, RNA sequencing analysis also suggests an overall increase in age-related heterogeneity. 

It should also be noted that these results may be biased toward the cortex region, as the independent data sources mostly included samples from the cortex region.

Another limitation is related to use of bulk RNA expression datasets, where each value is an average for the tissue. While it is important to note that our results indicate increased heterogeneity between individuals rather than cells, the fact that the brain is composed of different cell types raises the question if increased heterogeneity may be a result of changes in brain cell-type proportions. To explore the association between heterogeneity and cell-type specific genes, we used FACS-sorted cell type specific transcriptome dataset from mouse brain [@Cahoy2008]. We only had nine genes that have consistent heterogeneity increase and are specific to one cell-type. Eight out of nine were highly expressed in oligodendrocytes. However, we did not observe any significant association between cell-type specific genes and heterogeneity (Figure \@ref(fig:hetFigS23), \@ref(fig:hetFigS33)). 

### Conclusion

In the first study (Section \@ref(veronikaresults)), our comparison showed that the correction strategy plays a pivotal role in identifying the specific set of differentially variable (DV) genes. However, irrespective of the approach and correction method used, a transcriptome-wide increase in the gene expression heterogeneity was observed, *i.e.* more genes showed a tendency to increase than to decrease expression heterogeneity with age. We also showed that most of the functional processes were susceptible to the ageing-related increase in the expression heterogeneity. 

In our second study (Section \@ref(ulasresults)), by performing a meta-analysis of transcriptome data from diverse brain regions we found a significant increase in gene expression heterogeneity during ageing, compared to development. Increased heterogeneity was a consistent pattern among diverse brain regions in ageing, while no significant consistency was observed across development datasets. Our results support the view of ageing as a result of stochastic molecular alterations, whilst development has a higher degree of gene expression regulation. Similar to our first study, we observed a widespread heterogeneity in genes and functional categories. However, the genes showing a *consistent* increase in heterogeneity during ageing are involved in pathways important for ageing and neuronal function. Therefore, our results demonstrate that increased heterogeneity is one of the characteristics of brain ageing and is unlikely to be only driven by the passage of time starting from developmental stages.
