# The link between ageing and age-related diseases {#UKBBChapter}

\small

**Data Availability**

*Supplementary Tables are available to download __[here](https://drive.google.com/file/d/1j6AJfTOD9euHCdZTlhtWYn22M1KdaQ5A/view?usp=sharing)__. At the moment the data is not publicly available but requires a permission which is granted case by case. All supplementary tables and summary statistics will be publicly available in the BioStudies database under accession number [S-BSST377](https://www.ebi.ac.uk/biostudies/studies/S-BSST377) upon publication of this work. Data tables are referred with the corresponding file names throughout the text. This research has been conducted using the UK Biobank Resource (application no. 30688).*

\normalsize

## Introduction

Risk of many age-related diseases is influenced by genetic variation. Genome-wide association studies (GWAS) have identified genetic variants that alter complex traits. Pleiotropy, where variants or genes influence multiple traits, is more prevalent than previously thought [@Bulik-Sullivan2015; @Cortes2020; @Cross-Disorder_Group_of_the_Psychiatric_Genomics_Consortium2013; @Pickrell2016], and it indicates that different traits share common causal pathways [@Solovieff2013]. Pleiotropy within the disease classification system [@Cortes2020] and in certain disease classes, such as immune-related diseases [@Ellinghaus2016; @Parkes2013] and cancer [@Bien2019], have been studied, but understanding of pleiotropy in age-related diseases (ARD) more broadly is limited. @Johnson2015 manually curated a list of 39 ageing-related biomarkers and diseases, collected their genetic associations from previous studies, and asked if they share commonalities. Although the number of genes implicated in diverse traits was low, they found that traits shared common pathways, such as NSN (*i.e.* nutrient-sensing network) and proteostasis [@Johnson2015]. Other studies aimed to understand if ageing and age-related diseases share common genes. Specifically testing if known ageing-related genes and their interactors are implicated in late-onset diseases, they showed significant but limited number of genes [@Fernandes2016; @Wang2009]. Despite the challenges of combining data from published studies, these studies provided the first hints that at least some late-onset diseases share common pathways [@Johnson2015], and they are related to some ageing-related genes [@Fernandes2016].

Genome-wide germline variations that increase the risk of diseases in old age may not be pruned by natural selection, or may be associated with beneficial phenotypes earlier in life. Indeed, comparing genetic associations in the GWAS-Catalog [@Buniello2019], @Rodriguez2017 found support for both theories.

### Research objectives

In this chapter, I present the investigation of causal connections between different ARDs by clustering them according to their age-of-onset, and determining whether the resulting clusters share common genetic risk variants. The UK Biobank (UKBB) [@Bycroft2018] includes genetic and health-related data for almost half a million participants. We extracted age-of-onset profiles for 116 diseases and identified unbiased clusters to define the relationship between disease incidence and age. We identified variants associated with each disease and compared the genetic associations between diseases based on the clusters. In particular, we aimed to answer the following questions:

* Do diseases follow similar age-of-onset profiles? 
* Are diseases with the same age-of-onset profile genetically similar? Can this be explained by co-morbidity and disease cause-effect relationships, or does it suggest a common aetiology?
* Does the genetic component shared in late-onset diseases overlap with known ageing-related genes and pathways? 
* Is the shared component druggable? 
* Do variants associated with different age-of-onset profiles support evolutionary theories of ageing?

## Exploratory analysis of the UKBB for ageing and age-related disease research

In this study, we used self-reported diseases and age-at-diagnoses in the UKBB [@Bycroft2018]. Using the samples that pass quality control (n = 484,598), we first performed an exploratory analysis using the basic demographics, disease data, and ageing-related data fields. 

There were more females (n=262,758) than males (n = 221,840) (Figure \@ref(fig:disFigS1)a). The age range of participants during the first visit was between 37 (minimum age of males = 37, females = 39) to 73 (maximum age of males = 73, females = 71) with a median value of 58 (median age of males = 58, females = 57) (Figure \@ref(fig:disFigS1)b). There were 13,697 participants who died after participating in the study and the death rate was higher in males (Figure \@ref(fig:disFigS1)c). As expected, height, weight, and BMI also differed in females and males (Figure \@ref(fig:disFigS1)d-f). 

Participants were also asked how they rate their health, how satisfied they are with their health, smoking status, alcohol drinker status, if other people generally say they look i) younger than they are, ii) about their age, or iii) older than they are, and if they had a close relative who had non-accidental sudden death. Overall, more people rated their health high and were happy with their health (Figure \@ref(fig:disFigS2)a-b). Most of the UKBB participants either never smoked or were previous smokers (Figure \@ref(fig:disFigS2)c) and are current alcohol drinkers (Figure \@ref(fig:disFigS2)d). Most of the participants also reported that people generally think they are either younger than their age or about the same age (Figure \@ref(fig:disFigS2)e). Most of the participants did not have any close relatives who died suddenly from non-accidental causes (Figure \@ref(fig:disFigS2)f). 

```{r disFigS1,out.width = '100%', fig.cap=c("Participant data in the UKBB after quality control steps. a) The number of female and male participants, b) Age distribution when participants first attended the UKBB assessment centre and answered self-reported questions, c) Age at death (every 5 years are binned together) for the participants who died after attending the UKBB assessment centre. The values are corrected for the number of female and male participants who passed the ages specified in the y-axis, d) Distribution of 'standing height' field in the UKBB, e) Distribution of 'weight' field in the UKBB, f) Distribution of BMI field calculated using 'standing height' and 'weight' fields in the UKBB."), fig.scap="Participant data in the UKBB after quality control steps."}
knitr::include_graphics(path = "figures/disease/FigureS1.png")
```

We also checked the distribution of other ageing-related fields, namely parents' age at death, age at menarche, and age at menopause. There were 391,842 participants whose at least one parent is dead. The distribution of age at death for parents was wide (10 to 117), but the majority of the data (between the first and third quantiles) lie between 65 and 79.5 (average age at death) (Figure \@ref(fig:disFigS3)a). The age at menarche differed between 5 and 25, with a median of 13 (Figure \@ref(fig:disFigS3)b). The age at menopause differed between 18 and 68, with a median of 50 (Figure \@ref(fig:disFigS3)c).

The number of self-reported operations ranged between 1 to 32, with a median of 1 and the number of self-reported medications ranged between 0 to 48, with a median of 2 (Figure \@ref(fig:disFigS4)a). Among 39,910 participants with cancer, most of them had only one cancer, while there was also a participant with 6 cancers (Figure \@ref(fig:disFigS4)b).

```{r disFigS5,out.width = '100%', fig.cap=c("Pairwise correlations between traits. Each row and column show a trait in the UKBB, and the colour shows the pairwise Spearman correlation coefficients between traits. Dark red denotes a strong positive correlation, while darker blue indicates strong negative values. Traits are ordered based on the hierarchical clustering of the correlation coefficients."), fig.scap="Pairwise correlations between traits."}
knitr::include_graphics(path = "figures/disease/FigureS5.png")
```

We then checked the correlations between these traits (Figure \@ref(fig:disFigS5)). As expected, age when attended assessment centre was very strongly correlated with age at death. It also showed a correlation with parental age at death, the number of non-cancer diseases, and the number of medications taken. 'Overall health rating' and 'health satisfaction' were also correlated with the number of diseases and medications. Moreover, these values also show a correlation to BMI and weight. While 'sex' and 'standing height' are correlated with 'weight', they are not correlated with 'BMI' and 'overall health rating' which are both correlated with 'weight'. BMI is also correlated with 'number of medications taken'.  

## Self-reported diseases in the UKBB

The UKBB includes disease information from two sources: i) disease ICD10/9 codes based on hospital episode statistics (HES) and ii) the self-reported (SR) diseases. We analysed the SR diseases as the participants also inform the age at diagnosis. And since they report the diseases they had at earlier ages as well, this data is less biased by the age distribution of the UKBB participants. Moreover, a previous study using the UKBB suggested that GWAS using self-reported diseases and ICD-10 codes were sufficiently similar [@Cortes2018]. 

```{r disFigS6,out.width = '100%', fig.cap=c("Disease hierarchy for the 116 diseases included in the analysis. The nodes are coloured by the disease categories as indicated in the legend."), fig.scap="Disease hierarchy for the 116 diseases included in the analysis."}
knitr::include_graphics(path = "figures/disease/FigureS6.png")
```

Like ICD10 codes, the UKBB SR Diseases are defined in a hierarchical structure (Figure \@ref(fig:disFigS6)). This tree is constructed by the UKBB nurses and it mostly reflects the system or the tissue that disease is most symptomatic in. Participants enter SR disease data with a trained nurse, who guides them. However, we observed that some participants did not consider the disease hierarchy while some did. For example, some patients having 'essential hypertension' also reported having 'hypertension' which is the parent node, while some did not. In order not to bias data, we propagated disease data towards upper levels, so that a participant with a disease at a lower level is always annotated with the connected nodes at upper levels. 

Importantly, we only considered 116 non-cancer diseases with at least 2,000 cases and that are not sex-specific. Although we exclude single-sex diseases, we included the ones that are more prevalent in females (thyroid problem, hypothyroidism, bone disorder, osteoporosis) or in males (abdominal hernia, gout, heart attack) (Figure \@ref(fig:disFigS7)). 

```{r disFigS7,out.width = '100%', fig.cap=c("Sex-stratified statistics for 116 selected diseases. (a) The distribution of the number of self-reported diseases (y-axis) stratified by sex (x-axis). (b) The distribution of disease prevalence in males and females. The x- and y-axes show the number of cases in 1,000 males and females (in log scale), respectively. The colour of each point denotes if a disease has a higher prevalence in females (rosy brown, above the dashed line) or males (slate grey, below the dashed line). The linear regression line is depicted as blue. Diseases having a residual value bigger than 3 standard deviations are labeled but not excluded as they are also common in the other sex."), fig.scap="Sex-stratified statistics for the 116 diseases."}
knitr::include_graphics(path = "figures/disease/FigureS7.png")
```

## Disease co-occurrences

We next calculated disease co-occurrences, using relative risk score to calculate associations and $\phi$ values as a measure of robustness [@Gutierrez-Sacristan2018; @Jiang2018; @Park2009; @Sanchez-Valle2018]. 

```{r disFigS8,out.width = '100%', fig.cap=c("Disease association matrix summarizing relative risk scores and correlations. Each row and column denote diseases, ordered by hierarchical clustering of risk scores. The colour is defined by relative risk scores while the size is determined by ɸ value, indicating the robustness. The diagonal tiles are coloured by the UKBB's disease hierarchy to visualise if diseases from the same category cluster together. Associations for the 62 diseases that have at least one relative risk ratio higher than four or lower than minus four ($|log_2RR| ≤ 2$) are plotted."), fig.scap="Disease association matrix summarizing relative risk scores and correlations."}
knitr::include_graphics(path = "figures/disease/FigureS8.png")
```


There were five major clusters with high relative risk scores and robustness and they seemingly cluster by disease categories: i) musculoskeletal/trauma diseases and early-onset gastrointestinal diseases such as appendicitis, ii) other musculoskeletal/trauma diseases such as sciatica and disc problems, iii) respiratory/ENT diseases such as bronchitis and pneumonia, iv) cardiovascular diseases and diabetes, v) retinal problem, glaucoma, and cataract (Figure \@ref(fig:disFigS8)). While most of these clusters are biologically plausible, some could be explained by reporting bias, *e.g.* bone fractures and appendicitis. Moreover, we saw a strong negative correlation between osteoarthritis and arthritis (nos). The disease 'arthritis (nos)' does not include osteoarthritis by definition (nos = not osteoarthritis) and seeing this association suggests that we can detect co-occurrences reliably. 

## Age-of-onset clusters

In order to characterise diseases based on their age-relevance, we used age at diagnosis as a proxy to disease onset and derived disease age-of-onset profiles (Figure \@ref(fig:disFigS9)-\@ref(fig:disFigS18)). On average, cardiovascular and endocrine diseases had a high median age-of-onset, while infections had the lowest (Figure \@ref(fig:disFigS19)). 

```{r disFigS19,out.width = '100%', fig.cap=c("Distribution of median age-of-onset (y-axis) across categories (x-axis). Points show diseases, grouped by the categories (individual boxplots). Categories are ordered by the median value of the median age-of-onset."), fig.scap="Distribution of median age-of-onset across categories."}
knitr::include_graphics(path = "figures/disease/FigureS19.png")
```

We then clustered diseases into 4 clusters (the optimum number determined by gap statistic) using the PAM algorithm and disease dissimilarities calculated using CORT (temporal correlation measure) distance [@Chouakria2007] (Figure \@ref(fig:disFig1), Supplementary Table S1). 

```{r disFig1,out.width = '100%', fig.cap=c("Age-of-onset profiles clustered by the PAM algorithm, using dissimilarities calculated using (CORT). The y-axis shows the number of individuals who are diagnosed with the disease at a certain age, divided by the total number of people having that disease. Values are calculated using the median values for 100 permutations of 10,000 people in the UKBB. The x-axis shows the age-of-onset in years. Each line shows one disease and is coloured by disease categories. The heatmap on the right upper corner shows the percent overlap between categories and clusters. Numbers give the \\% of an age-of-onset cluster belonging to each category."), fig.scap="Age-of-onset clusters."}
knitr::include_graphics(path = "figures/disease/Figure1.png")
```

Cluster 1 diseases (n = 25) showed a rapid increase with age after the age of 40; 11 were cardiovascular diseases, but this cluster also included other diseases such as diabetes, osteoporosis, and cataract. Cluster 2 (n = 51) diseases started to increase in the population at an earlier age of 20, but had a slower rate of increase with age; diseases in this cluster were the most diverse, including 17 musculoskeletal, 13 gastrointestinal diseases, as well as others such as anaemia, DVT, thyroid problems, depression. Cluster 3 diseases (n = 30) showed a low age dependency with a mostly uniform distribution across ages, but with slight increases around the ages of 10 and 60 years. This category included similar numbers of immunological, neurological, musculoskeletal, gastrointestinal and respiratory diseases but all have an 'immune' component even if not classified in this way by the UKBB (*e.g.*, inflammatory bowel disease (gastrointestinal), asthma (respiratory), psoriasis (dermatology)). Cluster 4 (n = 10) had a peak at around 0-10 years of age and included respiratory diseases (n = 5) and infections (n = 4). Notably, all infectious diseases were in this cluster.

## Genetic similarities between diseases 

Using linear mixed models implemented in BOLT-LMM [@Loh2015], we performed GWAS on each disease separately and included approximately 10 million common variants that pass quality control. Considering associations with the literature standard p-value lower than $5*10^{-8}$ as significant [@Panagiotou2012; @Peer2007], we next quantified the associations for each disease, category, and age-of-onset cluster (Figure \@ref(fig:disFigS20)). The major histocompatibility complex (MHC) region is excluded from all analyses, as in the literature, because of its unusually high effect sizes and LD patterns (chr6: 28,477,797 - 33,448,354) [@Bulik-Sullivan2015; @mhc2020]. Out of 116 diseases, 36 had no significant association and the total number of polymorphic regions with at least one significant association was 93,817. The maximum number of significant associations was 35,001 (hypertension) and the median and mean were 13.5 and 1389.3, respectively. We also checked if diseases from different age-of-onset clusters vary in the number of associations. Cluster 4 had almost no significant associations (the disease with the maximum number of associations had only 3 significant variants). Although cluster 1 had the highest number of significant associations on average, the values across clusters 1, 2, and 3 were not significantly different (Figure \@ref(fig:disFigS20)b). Moreover, endocrine, immunological, cardiovascular diseases had the highest number of associations and infections had the lowest (Figure \@ref(fig:disFigS20)c). Only 1% of the significant polymorphisms (n=932) were in coding regions, and of these 49% (n=452) were missense and only 1% (n=10) were nonsense. We further found that 47% of significant variants (n=43,810) were associated with multiple diseases, but only ~9% were associated with multiple diseases from different categories (n=8,048) and again ~9% with different age-of-onset clusters (n=8,801) (Figure \@ref(fig:disFigS21)).

```{r disFigS20,out.width = '100%', fig.cap=c("(a) Number of diseases for different number of significant variants (p≤5e-8). Diseases with the highest number of associations (N≥10,000) are given as an inset table. (b) Comparison of the number of significant associations (y-axis, in log scale) across age-of-onset clusters (x-axis) (ANOVA after excluding cluster 4, p = 0.06). Since the y-axis is in log scale, diseases with zero significant associations are not shown on the graph. (c) The same as (b) but for disease categories. Categories are ordered by the median number of significant SNPs."), fig.scap="Distribution of significant disease associations across diseases, age-of-onset clusters, and disease categories."}
knitr::include_graphics(path = "figures/disease/FigureS20.png")
```

```{r disFig2,out.width = '80%', fig.cap=c("(a) Network representation of the genetic similarities. Nodes (n=47) show diseases with at least one significant genetic similarity and are coloured by the age-of-onset cluster. Edges (n=167) are weighted by the genetic similarity corrected by disease categories and co-occurrences. (b) The difference between genetic similarity within and across the age-of-onset clusters. The y-axis shows genetic similarity corrected by category and co-occurrence on the log2 scale and x-axis groups similarities into different or same age-of-onset clusters. c) Network representation of the cause and effect relationships between diseases. Each node (n=48) shows a disease, coloured by the age-of-onset cluster. Size of the nodes represent the number of significant causal relationships. Arrows show the cause and effect relationship between pairs with FDR corrected p≤0.01 and GCP>0.6. The inset bar plot shows the percent significant causal relationships (y-axis) between disease 1 (x-axis) and disease 2 (bars coloured by the age-of-onset)."), fig.scap="Genetic similarities and cause-effect relationships between diseases."}
knitr::include_graphics(path = "figures/disease/Figure2.png")
```

We next sought to characterise the genetic similarities between diseases using a score that shows the excess of overlapping associations between diseases, given the number of significant associations for each disease. Importantly, we calculated genetic similarities between 80 diseases that have at least one significant association, excluding the pairs that are vertically connected (*i.e.* ancestors to child) in the disease hierarchy (*e.g.* essential hypertension and hypertension). We found 47 significant overlaps and diseases with similar age-of-onset profiles showed a higher genetic similarity, even when controlled for disease categories and co-occurrences (F-test p = 1.19 x 10-8, Figure \@ref(fig:disFig2)a-b,\@ref(fig:disFigS22)). Moreover, this trend was reproducible when each cluster was analysed separately (Figure \@ref(fig:disFigS23)). We repeated the analysis using 1,703 previously defined LD blocks [@Berisa2016] instead of taking all SNPs as independent, and found significant similarity only between diseases of the same age-of-onset cluster (Figure \@ref(fig:disFigS24)). Importantly the disease categories (p=0.89) and co-occurrences (p=0.15) did not explain the similarity.

## Cause-effect relationships between diseases

Using a recent methodology developed by @OConnor2018, we tested for partial or fully causal relationship between diseases. In particular, the method identifies if a latent causal variable (LCV) mediates the genetic correlation between diseases. Using a genetic causality proportion, it assigns a causal relationship if one of the diseases is more strongly correlated with the LCV. The authors report that, unlike mendelian randomisation, this method can distinguish between the correlation due to common aetiology and causation. We tested for potential causation between 60 diseases, excluding the ones with less than 10 significant genetic variants and low heritability estimates ($Z_h<7$) [@OConnor2018]. Also, similar to genetic similarities, we did not calculate the causation between diseases that are vertically connected in the disease hierarchy. Following the same significance criteria proposed in the methods article (FDR corrected p≤0.01 and mean $GCP>0.6$ - *i.e.* genetic causality proportion), we found significant evidence for fully or partial genetic causality in 91 disease pairs between 48 diseases (Figure \@ref(fig:disFig2)c, Supplementary Table S2). DVT, venous thromboembolic disease, and pulmonary embolism had the highest number of out degrees, meaning they were found as causal for multiple diseases, including all 3 age-of-onset clusters and 5 different disease categories. Gastro-oesophageal reflux (GORD)/gastric reflux and oesophageal disorder, on the other hand, had the highest number of in degrees, meaning there are multiple diseases detected as causal. A likely explanation is that there are multiple types of drugs, used for different conditions, that are associated with increased risk or exacerbated symptoms of GORD [@Mungan2017]. These causal diseases spanned 5 disease categories and age-of-onset clusters 1 and 2.

Using Fisher's exact test, we also tested if the causal relationships were more common between diseases in certain age-of-onset clusters but did not find any significant difference (FDR corrected p>0.1 for all comparisons, inset bar plot in Figure \@ref(fig:disFig2)c). Thus, we concluded that although there are some causal links, the high genetic similarity within clusters (Figure \@ref(fig:disFig2)a-b) cannot be explained by causation but more likely to include common aetiologies.

## Known ageing-related genes and genes associated with different age-of-onset clusters

We next mapped all variants to genes based on proximity or known eQTLs using the GTEx data [@Gamazon2018]. To assess the reproducibility of the genes identified from this study, we compared the significant hits we found with all those reported in the GWAS Catalog. We verified that most of the diseases had significant overlaps with the same or associated traits in the GWAS-Catalog (*e.g.* osteoporosis and bone density), confirming our results are reproducible with independent data (Supplementary Table S3). We next compiled the genes associated with multiple diseases and multiple categories and grouped them based on the age-of-onset cluster of the associated diseases (Supplementary Table S4). In particular we created two sets of genes, 'multidisease' and 'multicategory', for clusters 1, 2, and 3. We exclude cluster 4 from downstream analysis, because the number of variants significantly associated with this cluster was low (n=7 associated with 5 diseases), mapping to only 2 genes (*ZPBP2*, *NPC1L1*). We also compiled genes associated with multiple diseases or categories in combinations of different age-of-onset clusters. Importantly, genes associated with multiple clusters are not in the gene sets for individual clusters as they only include genes specific to those clusters, *e.g.* cluster 1, cluster 2 and cluster 1 & 2 sets all include mutually exclusive sets. 

```{r disFig3,out.width = '100%', fig.cap=c("a) Overlap between known ageing-related genes in databases and genes associated with diseases in different clusters. x-axis shows log2 enrichment score, and y-axis shows the age-of-onset clusters. The numbers of genes in each cluster (for both Multidisease and Multicategory genes) are given. The size of the points shows the statistical significance (larger shows marginal p-value≤0.05) and the colour shows different databases. The coloured numbers near the points show the numbers of overlapping genes. b-f) Gene Ontology (GO) Enrichment results for genes associated with diseases in b) Cluster 1, c) Cluster 2, d) Cluster 3, e) Cluster 1 and 2, f) Cluster 1, 2 and 3. Representative GO categories for significantly enriched categories (BY-adjusted p-value ≤ 0.05) are listed on the y-axis. Log2 enrichment scores are given on the x-axis. The colour of the bar shows the result for multidisease and multicategory genes. There was no significant enrichment for cluster 1 \\& 3 and 2 \\& 3."), fig.scap="Enrichment analyses for the genes associated with different age-of-onset clusters and ageing-related genes or gene ontology categories."}
knitr::include_graphics(path = "figures/disease/Figure3.png")
```

We next sought to understand if the genes we identified were previously associated with ageing, by comparing the gene lists we compiled from the literature-based ageing databases: GenAge human (genes associated lifespan in humans or closely related species), human orthologs in GenAge model organism (genes modulating lifespan in model organisms), CellAge (genes regulating cellular senescence), DrugAge targets (drugs modulating lifespan in model organisms), and all databases combined [@Avelar2019; @Barardo2017; @Tacutu2018] (Figure \@ref(fig:disFig3)a). In general, genes associated with clusters 1 and 2 showed significant enrichment with known ageing-related genes, but not cluster 3. The list of overlapping genes is given in Supplementary Table S5. CellAge database showed the largest number of significant overlaps, with genes associated with clusters 1, 2, and both 1 and 3. DrugAge targets had a significant overlap with clusters 1, 2, and both 1 and 2. GenAge Human only had significant association with genes associated with both cluster 1 and 2. GenAge model organism data significantly overlapped with genes associated with both cluster 1 and 2, and all clusters (1 & 2 & 3). The results showed that, although the number of overlaps is low, the clusters 1 and 2, constituting age-associated diseases, share a significant genetic component with known ageing-related genes, while cluster 3 did not.

## Biological functions of the genes associated to different age-of-onset clusters

We next did a Gene Ontology (GO) enrichment analyses using Biological Process (BP), Molecular Function (MF) and Cellular Component (CC) categories. Cluster 1 was associated with many lipoprotein-related categories, cellular signaling, cellular response, cell cycle arrest, and blood pressure (Figure \@ref(fig:disFig3)b). Cluster 2 showed association to MHC class II binding, fibrinolysis, and negative regulation of epithelial cell (Figure \@ref(fig:disFig3)c). Cluster 3 had associations to many immune-related categories and cell adhesion (Figure \@ref(fig:disFig3)d). Genes associated with both cluster 1 and cluster 2 were related to nucleosome complex, glucose homeostasis, retinoic acid binding (Figure \@ref(fig:disFig3)e). Genes in clusters 1 and 3, and in 2 and 3 did not have any significant associations. Genes associated with at least one disease in all clusters were related to nucleosome complex, interleukin-7 response, differentiation, telomere, and gene silencing (Figure \@ref(fig:disFig3)f). Here we listed the categories that are representative to all other significant associations. The full list is given in Supplementary Table S6, and the procedure of selecting representatives is described in Section \@ref(disGO).

## Drug repurposing to improve health-span


```{r disFigS25,out.width = '100%', fig.cap=c("Drug-target gene interaction network for the drugs that specifically target multicategory cluster 1, cluster 2 or cluster 1 and 2 genes as determined by Fisher's exact test. Blue diamonds show the drugs with significant association or targeting only one gene in these gene groups. Circles represent the genes targeted by the significant hits, coloured by their age-of-onset cluster. Grey circles show the genes targeted by these drugs but are not among the gene set of interest. The interaction data is collected from DGIdb."), fig.scap="Drug-target gene interaction network for the drugs specific to multicategory age-dependent diseases."}
knitr::include_graphics(path = "figures/disease/FigureS25.png")
```

Identification of drugs that can target the multicategory genes in clusters 1 or 2 poses a possibility to treat many diseases simultaneously and thus extend health-span in the elderly. Thus, we investigated if there are drugs that target these genes specifically (p≤0.01 or having only one specific target, Figure \@ref(fig:disFigS25)). We found drugs targeting multicategory cluster 1 genes i) *ABCC8* and *KCNJ11*, which code for parts of K-ATP channels, ii) *CCND1*, iii) *MTAP*, iv) *NPC1L1.* There were also several drugs targeting multicategory genes associated with both cluster 1 and 2 diseases, such as *PPARG, INSR, FGFR4, MAPKAPK5, ALDH2, PTPN11*. Interestingly, 23 of the significant drugs were approved drugs for 14 conditions, including diabetes, hyperlipidaemia, neoplasms, osteoporosis, cardiovascular diseases (list of all drugs and indications available in Supplementary Table S7). Moreover, one of the drugs we identified, prunetin (targeting *ALDH2*), was previously shown to improve the lifespan of male Drosophila melanogaster [@Piegholdt2016].

## Evolution of ageing and age-related diseases 

```{r disFig4,out.width = '100%', fig.cap=c("Risk allele frequency distributions (y-axis) of different age-of-onset clusters (x-axis) in the UKBB for a) SNPs associated with one disease (excluding antagonistic associations), b) SNPs specific to one cluster (excluding antagonistic associations) (ns: p>0.05, *:p≤0.05, **: p≤0.01, ***: p≤0.001, ****: p≤0.00001), and c) SNPs that have antagonistic association with cluster 1 and 2 (excluding agonists between cluster 1 and 2). d) The same as panel c but for different 1000 Genomes super-populations (ALL: complete 1000 Genomes cohort, AFR: African, AMR: Ad Mixed American, EAS: East Asian, EUR: European, SAS: South Asian). The p-values specified on the plots are marginal p-values calculated for two-sided tests and are not corrected for multiple testing."), fig.scap="Risk allele frequency distributions of different age-of-onset clusters and SNPs specific to one disease, one age-of-onset cluster, or antagonistic between cluster 1 and 2."}
knitr::include_graphics(path = "figures/disease/Figure4.png")
```

We next sought to understand the abundance of disease-associated variants in the population and their relationship with the evolutionary theories of ageing. We first hypothesized that according to the mutation accumulation theory of ageing (MA), SNPs associated with later-onset diseases (Cluster 1) would have a higher frequency than the SNPs associated with diseases that occur at earlier ages (Clusters 2 & 3). We therefore compared the risk allele frequencies of diseases in different age-of-onset clusters. As SNPs which are close together in the genome are expected to have similar allele frequencies due to linkage, we calculated the median risk allele frequency for SNPs within previously defined LD blocks [@Berisa2016]. Diseases of cluster 1 had significantly higher risk allele frequencies than cluster 2, both for the SNPs associated with one disease (Figure \@ref(fig:disFig4)a, Wilcoxon test p = 0.00033) or with one cluster (Figure \@ref(fig:disFig4)b, Wilcoxon test p = 0.0068, also confirmed by bootstrapping n = 100 loci for B = 1,000; Figure \@ref(fig:disFigS26)). We also analysed the risk allele frequencies in populations of the 1000 Genomes Project [@1000_Genomes_Project_Consortium2015] data and confirmed that the same trend is observed in all super-populations (Figure \@ref(fig:disFigS27)-\@ref(fig:disFigS28)). 

To test the antagonistic pleiotropy theory (AP), we first asked if the diseases with different onsets have an excess of antagonistic SNPs. Here, following the literature [@Rodriguez2017], we defined a pleiotropic biallelic SNP as agonistic if the risk allele is the same for different diseases, and as antagonistic if opposite alleles are associated with increased risk for different diseases. If one of these diseases are under a stronger negative selection, then the risk allele of the other disease could increase with time. Comparing the proportion of agonist and antagonist SNPs within and between the age-of-onset clusters, we found that there is an excess of antagonistic pleiotropy between diseases with different age-of-onset profiles (Fisher's exact test p<0.001, Supplementary Table S8). Next, we tested the differences in risk allele frequencies between the clusters as AP predicts a higher risk allele frequency for late-onset diseases. Interestingly, the difference between the risk allele frequencies for cluster 1 and cluster 2 was not significant for the UKBB population (Figure \@ref(fig:disFig4)c). However, all 1000 Genome super-populations except for Europeans had higher risk allele frequencies for cluster 1 diseases (Figure \@ref(fig:disFig4)d). A potential explanation is that it is easier to detect antagonistic associations for SNPs with a minor allele frequency closer to 0.5 because of the increased power for both alleles. Indeed, associations with a larger effect size showed the expected differences in allele frequencies, although the number of independent loci was limited (Figure \@ref(fig:disFigS29)). We also examined the type of diseases and genes associated with antagonistic pleiotropy. The main driver of the pattern was the loci with *ABCG8* and *ABCG5* genes, showing antagonistic relationship for high cholesterol (cluster 1) and other lipid-related diseases that are in cluster 2, such as gallbladder disease and cholelithiasis. Another locus included variants that show antagonistic relationship with cardiovascular disease (cluster 1) and cluster 2 diseases gout (*ADH1B*), osteoarthritis and joint disorder (*SLC39A8*), and osteoarthritis (*BANK1*). Another potential candidate was a locus associated with hypertension (cluster 1) and musculoskeletal diseases (cluster 2), but this locus included multiple candidate genes (Supplementary Table S9).

## Methods

#### UK Biobank Data
Data is downloaded using bash and following the guidelines provided by the UK Biobank.

**Sample quality control**

After excluding all samples from individuals who have withdrawn their data from UK Biobank, we first filtered out all samples without genotypes (N = 14,248). Then, we used the following criteria for the remaining 488,295 samples.

**Discordant sex:** Data includes two entries for sex: 1) self-reported and 2) genetic sex determined using the call intensities on sex chromosomes. There are multiple reasons why these two entries may not correspond, such as sample mishandling, errors in data input, transgender individuals, and sex chromosome aneuploidies [@Anderson2010; @Bycroft2018]. Since we used sex as a covariate in our GWAS model, we preferred to be cautious about this issue and excluded all cases where the genetic sex and self-reported sex do not correspond and all cases where sex chromosome aneuploidy is detected. Specifically, we used the fields '31-0.0' (Sex) and '22001-0.0' (Genetic sex) to compile discordant information. There were 235 self-reported males being identified as female by the genetics, and 143 self-reported females being identified as males by the genetics. We excluded these 378 cases, making only 0.077% of the data. Moreover, field '22019-0.0' (Sex chromosome aneuploidy) is used to exclude cases with sex chromosome aneuploidy. There were 651 cases of aneuploidy, making 0.133% of all data. 181 of these cases (27.80% of aneuploidy cases) are also detected as discordant information in the first step. This corresponds to 47.88% of discordant sex cases. Overall, we identified 848 samples to be excluded based on this criterion.

**Genotype call rate & Heterozygosity:** For exclusions based on missingness and heterozygosity we only used the suggested exclusions by UK Biobank. Specifically, we used the field '22010-0.0' (Recommended genomic analysis exclusions) and determined the cases with 'poor heterozygosity/missingness' (N = 469). We next used the field '22018-0.0' (Genetic relatedness exclusions) and noted down the cases with 'Participant self-declared as having a mixed ancestral background' (N = 692), and the cases with 'High heterozygosity rate (after correcting for ancestry) or high missing rate' (N = 840). Lastly, there were 968 cases that are suggested as outliers for heterozygosity or missing rate, field '22027-0.0' (Outliers for heterozygosity or missing rate).  Genotype missingness and heterozygosity are widely used as a measure of DNA sample quality. We then checked the scatter plots for logit(Missingness) vs. Heterozygosity for each Ethnic Background, in accordance with the identification of samples to exclude by the UK Biobank [@Bycroft2018] (Figure \@ref(fig:disFigS31)). Logit transformation is used to linearize sigmoidal distribution of missingness. Investigation of heterozygosity can detect DNA sample contamination, inbreeding, or mixed ethnicity [@Anderson2010]. This quality check enables seeing that people with a mixed ethnicity tend to have a higher heterozygosity, even after correcting for PCs. We confirmed these are in accordance with the original article and excluded the samples suggested by the UK Biobank.

Overall, there were 3,697 samples excluded based on these two criteria. Please note that the numbers presented above may not add up to this number. That is because there were some samples excluded because of multiple criteria. The percent overlap across multiple criteria is given in Figure \@ref(fig:disFigS32).

After the exclusions, we had 484,598 samples.

**Preparing the trait data**

Using the samples that pass the quality control (N=484,598), we subset the data so that it includes only the baseline visit. Apart from the data that is already available in UK Biobank, we calculated some other values: 

1. BMI: Using the columns for 'Weight' and 'Standing height' we calculated BMI as: Weight / (Standing Height / 100)$^2$,
2. Parent Age at Death - Minimum: The youngest age at which either parent died. 
3. Parent Age at Death - Maximum: The age of death for the parent who lived longest. 
4. Parent Age at Death - Average: The average age of death for the two parents. If neither of the parents died, or if the data is unavailable, these values (2-4) are set to be NA. If only one parent died, we use the corresponding age as both the minimum, maximum, and average. 
5. The number of self-reported non-cancer diseases: The number of unique self-reported non-cancer illnesses each participant recorded in the baseline recruitment. 
6. The number of self-reported cancers: The number of unique self-reported cancers each participant recorded in the baseline recruitment. 
7. Self-reported diseases after taking the disease hierarchy into consideration (Propagated disease data): The self-reported diseases in UK Biobank are not independent, but rather are organised in a hierarchical manner. Using the relationship information between diseases, we propagate disease-participant associations, upwards, including terms higher up the tree. For example, if a person reports having "essential hypertension", we also annotate that person with "hypertension", and "cardiovascular disease". 
8. Age at diagnosis for the self-reported diseases after taking the disease hierarchy into consideration (Propagated age at diagnosis data): We re-defined age at diagnosis using the minimum age at diagnosis for all the diseases that are child term for a particular disease. 
9. The number of self-reported non-cancer diseases after taking the disease hierarchy into consideration (Propagated number of non-cancer diseases): The number of unique self-reported diseases each participant records after taking into account the data propagation. 
10. Age when the last deceased person died: We calculated the age of each person when the last death entry in the UKBB happened. This value is used to calculate the proportion of people who died at a certain age interval in Figure \@ref(fig:disFigS1)c.

**Selecting diseases to analyse**

We calculated the disease occurrences for all self-reported diseases in UK Biobank. Specifically, among the cohort we will use, we calculated how many participants and what proportion of males and females reported each disease. Since we analyse the same set of SNPs that have MAF>=0.01 across multiple diseases, to decrease the false positive rate in GWAS, we limited the diseases to a subset with at least 2,000 cases (n = 129 out of 472). Moreover, we only focused on diseases that are common and not sex-specific, *i.e.* we only considered diseases that are seen in 1 in every 1,000 males and females (n = 189 out of 472). The intersection of these two conditions was 116 diseases and we excluded all others.

In this study, we only analysed self-reported non-cancer diseases (field '20002') and did not combine self-reported cancers (field '20001'), mainly because i) the number of cases is low (45,224 compared to 384,906 for other diseases), ii) cancer is thought as a result of a complex interaction between germline and somatic mutations [@Kanchi2014; @Khurana2016], whereas the evidence for the effect of somatic mutations in other diseases is limited to rare and neurological disorders [@Poduri2013; @Zhang2018], iii) the relationship between cancer and ageing is complex, *e.g.* while telomere attrition and cellular senescence, are thought to be evolved as a tumour suppressor mechanisms; ageing-related changes in epigenomic landscape and genomic instability contribute cancer occurrence [@Finkel2007]. Thus, although a similar analysis using cancers would be interesting, we only focused on non-cancer self-reported diseases in this study.

#### Disease co-occurrences

```{r discooccurTable}
ds = data.frame(`.` = c('Disease A','Not Disease A'),
                `Disease_B` = c('Nab','Nnab'),
                `Not_Disease_B` = c('Nanb','Nnanb'),
                `Total` = c('Ta','Tna'))
kable(ds, format = "latex", booktabs = T,  caption = 'Contingency table for disease co-occurrence calculations.', longtable=T) %>%
    kable_styling(font_size = 12)
# kable(dsetinfo, format = 'pandoc', booktabs = T,  caption = 'Dataset information', longtable=T, font_size = 8)
```

**Relative risk (RR) score**

Relative risk is an estimate of having the disease A, when already affected by disease B. Overall it measures if disease A co-occurs with disease B more frequently than expected if these diseases were independent in the population. It is calculated as a fraction between the number of patients diagnosed with both diseases and a random expectation based on disease prevalence [@Sanchez-Valle2018]. Mathematically it can be expressed as follows, using the values from Table \@ref(tab:discooccurTable):

\begin{equation}
\begin{aligned}
    P_{exposed} = \dfrac{N_{ab}}{T_a}, P_{notexposed} = \dfrac{N_{nab}}{T_na} \\\\
    RR = \dfrac{P_{exposed}}{P_{notexposed}} \\\\
    CI = lnRR \pm 1.96 \sqrt{\dfrac{\dfrac{T_a-N_{ab}}{N_{ab}}}{T_a} + \dfrac{\dfrac{T_{na}-N_{nab}}{N_{nab}}}{T_{na}}}
  (\#eq:disRR)
\end{aligned}
\end{equation}

**$\phi$ value (Pearson correlation for binary variables)**

It measures the robustness of the association between diseases based on co-occurrences [@Gutierrez-Sacristan2018]. Mathematically, it can be expressed as:

\begin{equation}
    \phi_{AB} = \dfrac{C_{AB}N - P_AP_B}{\sqrt{P_AP_B(N-P_A)(N-P_B)}}
  (\#eq:disphi)
\end{equation}

$N$: the total number of individuals

$P_{A}$: Prevalence of disease A

$C_{AB}$: Number of patients with both diseases

$\phi$ ranges between -1 and 1, where the sign indicates the type of association.

#### Disease age-of-onset

**Disease dissimilarity measure**

**Temporal correlation:** In order to calculate dissimilarities among diseases, we use CORT [@Chouakria2007] distance as included in R package TSclust [@Montero2014]. Euclidean distance and dynamic time warping [@Berndt1994] are the two most widely used proximity measures for time series proximity. However, they are both calculated based on the closeness of the values and disregard the growth behaviour. Correlation-based measures are also used to calculate the similarity between time series. However, Pearson correlation overestimates the similarity because of the underlying temporal dependency and Spearman correlation fails to consider the growth rate as it is based on ranks. Chouakria et al., on the other hand, suggests a measure that also considers the proximity-based on growth behaviour, CORT. Temporal correlation between two time series objects S1=(u1,u2,...,up) and S2=(v1,v2,...,vp) is calculated as follows:

\begin{equation}
    CORT(S_1,S_2) = \dfrac{\sum_{i=1}^{p-1}{(u_{i+1}-u_i)(v_{i+1}-v_i)}}{\sqrt{\sum_{i=1}^{p-1}{(u_{i+1}-u_i)^2}}\sqrt{\sum_{i=1}^{p-1}{(v_{i+1}-v_i)^2}}}
  (\#eq:discort)
\end{equation}

CORT ranges between -1 and 1. A value of CORT = 1 implies that two time series increase or decrease simultaneously with the same growth rate, whereas a value of -1 shows the same growth rate but in the opposite direction. If the value is 0, it means there is no temporal correlation between the series.

**Dissimilarity Index:** The dissimilarity index suggested by Chouakria et al, is calculated based on an automatic adaptive tuning function and considers similarity based on both values and behaviour, *i.e.* the strength of monotonicity and closeness of the growth rates as calculated by CORT measure introduced in the previous section. They suggest a dissimilarity index D as follows:

\begin{equation}
    D(S_1,S_2)=f(CORT(S_1,S_2)).\delta_{conv}(S_1,S_2)
  (\#eq:disdissimilarity)
\end{equation}

Where $f(x)$ is an exponential adaptive tuning function:

\begin{equation}
    f(x) = \dfrac{2}{1+exp(kx)}, k≥0
  (\#eq:disexpadaptivetuning)
\end{equation}

As k increases, the contribution of behaviour increases. We use k = 2 and as a result behaviour (CORT) contributes 76.2% to D and values ($\delta_{conv}$) contribute 23.8%. For $\delta_{conv}$ we used conventional Euclidean distance.

**Clustering diseases by age-of-onset**

We clustered data using 'partition around medoids (PAM)' algorithm [@Kaufman1990] based on the distance measure calculated using the previous step. The aim of this algorithm is to minimise the average distance (based on any dissimilarity measure) between the objects and their closest selected medoid object. It works very similarly to k-means, except instead of defining arbitrary points as the means, it defines medoids among the objects. Thus, it can incorporate any distance measure instead of just using the mean distance between points (*i.e.*, euclidean distances). The algorithm first searches for k number of objects that represent the structure of the data (Here the number k is assumed to be known a priori but see the next section for the determination of k). After finding a set of k medoids, k clusters are constructed by assigning each observation to the nearest medoid. Overall, the goal is to find k representative objects such that the sum of dissimilarities of the observations to their closest representative is as small as possible. After each assignment, medoid and non-medoid data points are swapped and a cost (sum of distances of points to the new medoid) is calculated. If the total cost of configuration is decreased, then the new configuration is maintained, otherwise, it is reversed. We used 'pam' function in the 'cluster' package [@Maechler2019] in R to apply this algorithm.

**Choosing the optimum number of clusters**

The clustering algorithm we used, PAM, clusters data into k clusters, which is determined by the user. So, even if there is no real structure in data, as we increase the number of clusters, we can get more and more clusters. A potential way to decide on the number of clusters is using the gap statistic [@Tibshirani2001]. This value is calculated by comparing logarithm of the within-sum-of-squares and comparing it to averages from simulated data where there is no structure.

\begin{equation}
    WSS_k = \sum_{l=1}^{k}{\sum_{x_i\in C_l}{d^2(x_i,\overline{x_l})}}
  (\#eq:disWSS)
\end{equation}

$k$: number of clusters

$Cl$: objects in the l-th cluster

$\overline{x}$: the average point.

Calculating only WSS, however, is not enough as it would be minimised when each point has its own cluster. Thus, we use the gap statistic which suggests calculating $log(WSS_k)$ for a range of values of k and compare it to that obtained by WSS calculated based on simulated data. So, after WSS is calculated for various values of k, the algorithm involves generating B (we choose B=1,000) reference datasets, using Monte Carlo sampling from a homogeneous distribution and re-calculate WSS for all k values. Using these values gap(k) statistic is calculated:

\begin{equation}
\begin{aligned}
    gap(k) = \overline{l_k}-log(WSS_k)\\\overline{l_k} =\dfrac{1}{B}\sum_{b=1}^{B}{log(W^*_{kb})}
  (\#eq:disgapstat)
\end{aligned}
\end{equation}

If the clustering is good (WSS is small) we expect lk to be higher than $log(WSS)$. Thus gap statistic is mostly positive and we are interested in the highest value. @Tibshirani2001 suggests using the smallest k such that,

\begin{equation}
    gap(k)≥gap(k+1)-s'_{k+1}
  (\#eq:disgapstatincrem)
\end{equation}

where

\begin{equation}
\begin{aligned}
    s'_{k+1}=sd_{k+1}\sqrt{1+\dfrac{1}{B}}\\
    sd^2_k=\dfrac{1}{B-1}\sum_{b=1}^{B}{(log(W^*_{kb})-\overline{l_k})^2}
  (\#eq:dissk)
\end{aligned}
\end{equation}

Using this approach, we determined k = 4.

#### Genome wide association study

**Preparing the files required for GWAS**

**Fixing FAM files:** In UK Biobank FAM files, the column for 'phenotype' includes batch that is coded with characters. In order to use BOLT-LMM [@Loh2015], we updated all the entries in this column to numeric values [@Loh2017].

**'Remove' files for BOLT-LMM:** BOLT-LMM accepts a list of individuals to be removed from the analysis as an input. These files are called 'remove' files and are in the FAM format. We prepared these files for i) withdrawn samples (n = 51), ii) samples that failed the quality control (n = 3,779), iii) samples that have information in PLINK files but lack BGEN files (n = 968).

**Calculating the SNP statistics:** In order to apply a quality filter for SNPs, using PLINK [@Anderson2010], we calculated i) p-values for each SNP showing whether it deviates from Hardy-Weinberg equilibrium, and ii) Minor allele frequencies (MAF).

**SNP Quality Control:** We excluded SNPs that deviate from Hardy-Weinberg equilibrium (p ≤ 1e-6, n = 202,473) or with a minor allele frequency (MAF) smaller than 0.01 (n = 127,969). In total, we discarded 314,697 SNPs (Note that the numbers do not add up as these SNPs can overlap).

**Phenotype File:** We created a phenotype file that can be used as an input for BOLT-LMM, including the following fields: sex, age when attended assessment centre, calculated BMI, assessment centre, ethnicity, batch, first 20 PCs, and self-reported diseases (one column per disease).

**GWAS run using BOLT-LMM**

For each disease, we run GWAS separately using BOLT-LMM with the following inputs:

* We remove the samples that are in plink files but now in bgen; samples that did not pass our QC; samples from the individuals who have withdrawn their data from the UKBB
* We excluded the SNPs that deviate from Hardy-Weinberg equilibrium, and have minor allele frequency lower than 0.01.
* We used Sex, Age, BMI, assessment centre, ethnicity, batch, and the first 20 PCs as covariates.
* To run the mixed-model, a reference LD score table is required. We used LD scores generated using 1000G European-ancestry samples, which is provided with the BOLT-LMM download.
* Genetic map for hg19 file provided in the BOLT-LMM website.
* We set 'bgenMinMAF' argument to $10^-2$ and 'bgenMinINFO' parameter to 0.5 to only include SNPs that pass these criteria.

**GWAS Results**

We removed MHC region (chr6: 28,477,797 - 33,448,354) from the analysis and considered positions with a p-value lower than $5*10^{-8}$ as a significant association.

#### Coding Variants

We used VarMap [@Stephenson2019] to map variants to proteins and domains. VarMap provides detailed information about coding variants, including annotations for the missense, synonymous, and nonsense variations. In our analysis, if a variant is not annotated as a coding variant in VarMap output, we assumed it is non-coding.

#### Genetic similarities between diseases

In order to calculate the overlap between diseases we used the number of SNPs that are significantly associated with both diseases, but corrected by the number that is expected by chance, if two diseases are independent:

\begin{equation}
    Genetic Similarity = \dfrac{N_{common}}{N_{d1}*N_{d2}}*N_{total}
  (\#eq:disgensim)
\end{equation}

$N_{common}$: Number of SNPs in common.

$N_{dx}$: Number of SNPs associated with disease X.

$N_{total}$: Total number of SNPs analysed in the study.

The statistical significance of these genetic similarities is calculated using binomial test, and the similarity is only considered for downstream analysis if p<=0.01. Moreover, the value is only calculated if two diseases do not have any hierarchical relationships in the disease hierarchy.

In order to assess the genetic similarity within age-of-onset clusters, we further used linear regression to correct this value by disease co-occurrences (risk ratios) and disease categories (binary data showing whether two diseases are of the same category). The 'corrected genetic similarity' is the residuals from this linear model.

**LD Blocks**

In order to assess the similarity between different diseases we use overlaps across significant associations and thus preferred not to do fine mapping. However, a significant challenge is that genomic variations are not independent but instead linked in the genome. To understand the effect of linkage or overcome it, we made use of linkage disequilibrium blocks previously defined for human genome [@Berisa2016]. We repeated the analysis for genetic similarity after collapsing all positions within an LD block and thus creating independent genomic loci (n = 1,703). We use binary information for LD blocks, *i.e.* blocks with at least one significant association are considered as a hit, and the rest are not.

#### Analysis of causal relationships between diseases

Using the LCV method developed by @OConnor2018, we tested the causal relationships between diseases. We used the R functions developed by the authors and provided on GitHub (github.com/lukejoconnor/LCV/). We calculated the genetic causality proportion (GCP) between each disease pair, if the diseases have at least 10 significant variants and a significant heritability estimate as suggested by the developers (Zh≥7). We only calculated GCP if the diseases are not vertically connected on the disease hierarchy. Following the criteria applied by the developers, we considered pairs with FDR corrected p≤0.01 and mean GCP>0.6 as significant.

#### SNP to gene mapping

We map all SNPs analysed in GWAS to genes based on proximity and eQTL results.

**Using proximity**

Using VariantAnnotation [@Obenchain2014], GenomicRanges [@Lawrence2013], and TxDb.Hsapiens.UCSC.hg19.knownGene [@Carlson2015] packages in R, we mapped the genomic coordinates for each SNP to genes. Specifically, if a gene is within the coding region, intron, 5' or 3' UTR, or 1kb down- or up-stream of the transcription start site, we annotated that SNP to the gene. As a result, we had 4,443,872 SNP-gene associations for 4,236,176 SNPs and 22,228 Entrez gene IDs. We used the Ensembl biomaRt [@Durinck2009] package in R to retrieve HGNC symbols (17,994), Ensembl Gene IDs (20,507), and gene descriptions for the Entrez gene IDs obtained from TxDb.Hsapiens.UCSC.hg19.knownGene database.

**Using GTEx eQTL data**

Using SNP-gene associations based on GTEx v7 eQTL data (accessed on 04.09.2018) [@Gamazon2018], we associated SNPs with the genes they could potentially regulate. We generated a combined tissue list, which associates SNP to the gene if there is at least one tissue in which there is a significant (p <= 5e-8) association. As a result, there are 2,166,300 unique SNPs associated with 15,312 Ensembl Gene IDs. We used the biomaRt [@Durinck2009] package in R to retrieve HGNC Symbols (12,292), Entrez IDs (10,163), and gene descriptions.

**Comparison of proximity and eQTL based mapping**

Instead of only focusing on disease-associated SNPs, we first mapped all SNPs that we analysed to discover if there is a bias for certain genes (*e.g.* some genes could have many more SNPs because they are longer, or because they are already associated with certain traits and the chip is designed in that way). There were as much as 19,195 SNPs mapped to one gene (CSMD1) by proximity, whereas there were 82 SNPs per gene on average (median). The number of SNPs per gene was on average, higher for the mappings by eQTL (Figure \@ref(fig:disFigS33)a). The maximum was 8473 SNPs for HLA-C gene and the median number of SNPs per gene was 218. However, we did not consider MHC region in our downstream analysis and thus this region is also excluded. The correlation between the number of SNPs per gene was low (rho = 0.13, Figure \@ref(fig:disFigS33)b). Since the proximity based mapping is by definition dependent on the gene length, we also tested if there is a significant correlation between the number of SNPs per gene and gene length. While the correlation is low for gene mappings by eQTL (Spearman's correlation rho = 0.03, p = 1.073e-4), mappings by proximity show a high correlation as expected (Spearman's correlation rho = 0.87, p < 2.2e-16). This also explains the low correlation between eQTL and proximity-based mappings. We next checked the correlation between the number of SNPs per gene mapped by proximity but only to promoter region. The correlation between the number of SNPs and gene length decreased (rho = 0.21), and the correlation with the number of SNPs by eQTL slightly increased but was still low (rho = 0.08). Overall, we concluded that both eQTL data and proximity-based mapping could capture different information and decided to use both for the downstream analyses.

#### GWAS Catalog analysis

We accessed the GWAS Catalog on 30-07-2019 and used v1.0.2 e96 dataset [@Buniello2019]. We excluded all studies which used UK Biobank dataset (n = 190, data courtesy of GWAS Catalog team). Using the associations with a p-value lower than $5*10^{-8}$, we compiled significant associations between MAPPED_GENEs and MAPPED_TRAITs. We use GWAS catalog analysis to check if our GWAS hits are supported by previous studies and applied a Fisher test between all traits in GWAS catalog and the diseases in our study. p-values are corrected for multiple testing using FDR correction.

#### Analysis of the relevance with ageing

We downloaded GenAge human, GenAge model organism [@Tacutu2018] and DrugAge [@Barardo2017] data on Aug 13, 2019 and CellAge [@Avelar2019] data on Oct 02, 2019 (CellAge data is kindly provided by Avelar et al.). We used HGNC Symbols for GenAge and CellAge genes. In order to compile genes that are targeted by the drugs in DrugAge database, using the drug names in DrugAge data, we first compiled PubChem IDs using PubChem REST API [@Kim2019]. Using UniChem [@Chambers2013], we mapped PubChem IDs to ChEMBL IDs [@Gaulton2017]. Next, using DGIdb [@Cotto2018], we compiled the genes targeted by these ChEMBL IDs. As a result, we had 307 genes from GenAge human database, 902 genes from GenAge model organism database, 279 genes from CellAge database, and 714 genes targeted by DrugAge drugs. We next calculated the overlaps between these databases and the genes associated with multiple diseases or multiple categories in different age-of-onset clusters. To calculate the expected values and statistical significance, we used 10,000 permutations calculating the overlap for the same number of random genes among genes that can be detected by GWAS. Then, an odds ratio is calculated by dividing the observed value to the mean of expected values.

#### Functional Enrichment Test {#disGO}

Using the goseq package in R [@Young2010], which takes the gene length bias into account, we performed a functional analysis of the genes associated with different age-of-onset clusters. Using GO categories with more than 10 and less than 500 annotated genes, we applied an enrichment test for the Gene Ontology (GO) [@Ashburner2000; @The_Gene_Ontology_Consortium2019] Biological Process (BP), Molecular Function (MF), and Cellular Compartment (CC) categories. BY correction [@Benjamini2001] is applied to the p-values for all tests for all clusters and 3 GO Categories (BP, MF, and CC) combined. We considered associations with a BY-corrected p-value lower than 0.05 as significant. For the ease of visualisation and comprehension we selected representative categories for significant associations as follows: For each cluster and GO Ontology (*i.e.* BP, MF, CC) separately; i) Jaccard similarity index (*i.e.* number of genes in common divided by the number of unique genes in each category combined) is calculated between all significantly associated GO Categories; ii) Jaccard indices are hierarchically clustered and cut to k number of groups, where k is the minimum number of clusters which ensures median Jaccard similarity within a cluster is above 0.5; iii) The category with the highest average similarity to other categories in the same cluster is assigned as the representative.

#### Drug Repurposing

We searched for the drugs that specifically target multicategory genes in cluster 1, cluster 2, or cluster 1 and 2. Using the Fisher's exact test, we compiled the drugs in DGIdb [@Cotto2018] specifically targets these genes (p≤0.01) and drugs that target only one gene in one of these cluster. The interaction data is compiled from DGIdb, and the names of the drugs are obtained from ChEMBL REST API [@Gaulton2017].

#### Evolutionary Analysis

In order to test the mutation accumulation and antagonistic pleiotropy theories of ageing we used the risk allele frequencies in UK Biobank and 1000 Genomes super-populations [@1000_Genomes_Project_Consortium2015]. Risk allele is an allele that shows positive association with a disease. Since the SNPs are not independent and have similar allele frequencies in a given LD block, we analysed LD blocks instead of individual SNPs and used the median risk allele frequency for a given LD block. We used only the biallelic SNPs for this analysis. Allele frequencies for UK Biobank are calculated using BOLT-LMM and the allele frequencies for 1000 Genome super-populations are obtained from the vcf file provided on the 1000 Genomes project website. To test the antagonistic pleiotropy excess, we calculated the proportion of antagonistic vs. agonist SNPs within the same vs. different age-of-onset clusters using Fisher's exact test. We considered pleiotropic SNPs as agonist if the risk allele for two or more diseases are the same, and antagonist if the risk alleles are opposite. We only tested the risk allele frequency differences between cluster 1 and cluster 2. Also, we excluded any SNPs that are antagonistic within an age-of-onset cluster and agonist between clusters.


## Discussion

The number and the incidence of diseases increase with age. *Is this just a result of the accumulation of time-dependent stochastic damage, or is there a common genetic component among ageing-related diseases, which can also be attributed to the ageing process?* In this study, we aimed to answer this question by comparing genetic associations and age-of-onset distributions of 116 self-reported diseases in the UKBB. 

We first used an unsupervised, data-driven approach to classify diseases based on their age-of-onset profiles and found 4 main clusters (Figure \@ref(fig:disFig4)); diseases that i) rapidly increase after 40 years of age, ii) increase after 20 years of age, iii) do not show any age-related pattern, and iv) peak at around 10 years of age. Notably, unlike previous studies [@Fernandes2016; @Wolfson2009], using this unsupervised approach, we detect a distinction between cluster 1 and cluster 2, which show age-dependency but distinct age-of-onset distributions. 

We next found that the diseases in the same age-of-onset cluster show significantly higher genetic similarity, even when corrected for disease categories and co-occurrences (Figure \@ref(fig:disFig2)a-b). While correcting for the disease categories and co-occurrences, we may remove some true positive signals from the analysis. However, this correction is particularly necessary for our study, as we use the same cohort for multiple diseases, and thus, diseases vertically connected on disease hierarchy and those that co-occur uses the same set of samples. Nevertheless, we retain a significant signal even after this correction, suggesting diseases on average are more similar to others that have a similar age-of-onset profile. We next asked if cause and effect relationships, rather than a common aetiology, may explain this observation (Figure \@ref(fig:disFig2)c). We found 91 disease pairs with a significant causal association; however, there was no bias towards any specific age-of-onset cluster. Overall, our results suggest that even though each disease may have separate contributors, a genetic component may influence the age-dependency of diseases and the average age-of-onset. 

By mapping the significantly associated variants to genes, we defined pleiotropic genes that are specific to each cluster or combinations of clusters.  For example, *MTAP* and *CDKN2B* genes were associated with the highest number of diseases in cluster 1, spanning 3 disease categories; cardiovascular, endocrine, and eye diseases. *MTAP*, coding for an enzyme in polyamine biosynthesis, is found deleted in a number of cancers [@Pinero2019]. Being adjacent to a tumour suppressor gene, *CDKN2A*, its loss in cancer was attributed to being a bystander or regulator of *CDKN2A* expression [@Schmid1998]. Recent studies suggest *MTAP* itself is a tumour suppressor [@Kadariya2009; @Kadariya2013] and controls IGF1R activity [@Xu2019], which has been repeatedly shown to have evolutionary conserved relevance for ageing [@Lopez-Otin2013]. *CDKN2B* also codes for a tumour suppressor and is associated with multiple ageing-related diseases and parental longevity [@Pilling2017]. Notably, we did not analyse cancer associations in our dataset, and the diseases we analysed did not show a wide-spread co-occurrence with cancers that can bias the results (Figure \@ref(fig:disFigS30)). Although cancer associations are not included, finding these tumour suppressor genes is interesting; but potentially could be explained by the involvement of these genes in cell cycle regulation and cellular senescence. 

Genes associated with multiple diseases or multiple categories in clusters 1 and 2 were enriched among genes previously associated with ageing (GenAge, CellAge, DrugAge databases), but not cluster 3 alone (Figure \@ref(fig:disFig3)a). However, the association is established through a small subset of genes, as also previously described in the literature [@Fernandes2016; @Johnson2015]. Especially the genes in CellAge database, which include genes regulating cellular senescence in human cell lines, and genes targeted by the drugs in DrugAge, which include drugs modulating lifespan in model organisms, showed strong associations. Eleven genes associated with multiple categories and diseases spanning all three clusters had a significant association with the GenAge model organism genes, which modulates longevity in model organisms. Overall, both cluster 1 and cluster 2 showed significant association with known lifespan modulators. This suggests the age-associated diseases in clusters 1 and 2 share a genetic component with longevity determinants.

Our analysis so far suggested a potential role of regulation of cellular senescence in age-related diseases, but we also found several functional categories associated with different clusters. Cluster 1 genes were involved in lipoprotein-related functions, cellular response, and cell cycle arrest. Cluster 2, on the other hand, had a different functional profile, including epithelial cell apoptosis, blood coagulation and fibrinolysis, and MHC II protein binding. Cluster 3 had a strong immune-related profile, as we expect from the diseases in this category. The genes that are associated with both clusters 1 and 2 were related to the nucleosome, chromatin structure, homeostasis, metabolic process, and gene silencing. Interestingly, multicategory genes in Cluster 1 & 2 and 3, also had associations with the nucleosome, chromatin structure, and gene silencing. Since cluster 3 did not have an age-dependent profile, these categories could represent pleiotropic genes in general. Genes in this group were also associated with the interleukin-7 signaling pathway. Overall, these results suggest, although both cluster 1 and 2 genes were linked to previously identified ageing-related genes, they have distinct functional profiles. Moreover, the genes associated with diseases spanning these two categories were related to homeostasis and metabolic process, which were previously reported to change with age and modulate lifespan [@Lopez-Otin2013]. 

Therapies targeting the multicategory genes in cluster 1 and 2 diseases may alleviate the effects of multimorbidity at late ages and extend health-span. We found many drugs that are currently in use for different conditions, including age-related diseases we analysed, might be repurposed for other age-related conditions. Some of these drugs are already considered for multiple diseases from different categories. For example, acetohexamide, which targets the K-ATP channel, is in use for diabetes mellitus and is undergoing clinical trials for cataracts [@Acetohexamide2020].

*Can we use SNP-disease associations to understand the evolution of ageing?* Extending the work of @Rodriguez2017, we tested the two most widely appreciated evolutionary genetic theories of ageing (see Section \@ref(intrononprogAgeing)). In line with the MA theory, allele frequencies of the variants associated with ageing-related diseases were higher than the frequencies of cluster 2 disease variants (*i.e.*, diseases that start as early as the age of 20). Variants associated with cluster 3, which includes immune-related diseases, were not significantly different than cluster 1, although cluster 3 diseases can occur even at an earlier age. The genes in this cluster were also enriched in immune response categories, which were previously suggested to be under long-term balancing selection in humans [@Bitarello2018], explaining higher minor allele frequencies. We then showed that AP between different age-of-onset clusters is more common than expected. Comparing the allele frequencies of antagonistic variants between cluster 1 and 2 diseases in the UKBB and 1000 Genomes super-populations, we found support for the AP theory (*i.e.*, higher risk allele frequency for cluster 1 variants) only in non-European populations but not in the UKBB or European population. A potential explanation why we do not see the same trend in the UKBB and European population is that the probability of detecting significant association for two diseases associated with different alleles is higher for SNPs with a minor allele frequency closer to 0.5. Indeed, repeating the analysis with different effect size cutoffs (Figure \@ref(fig:disFigS29)), we saw that stronger associations had a clear risk allele frequency difference in these populations as well. Importantly, the number of independent loci is minimal, and thus, although there is a signal supporting the AP theory, it is unlikely that the effect is genome-wide. 

Overall, we showed that diseases clustered by their age-of-onset profiles show higher genetic similarity and are associated with specific genes, functions, and evolutionary characteristics. *How may these genes and functions affect whether a disease is an adult- or late-onset disease?* Here, we analysed the associations of the germline mutations, and thus, to understand their effects on the age-of-onset, we should consider their dynamic interaction with the environment, time-dependent somatic mutations, and the age-related changes in their expression. Somatic mutation load increases with age across almost all tissues, but the mutation profiles are tissue-specific [@Garcia-Nieto2019]. Thus, if somatic genetic mutations lead to the increased risk of disease, one may expect to see a clustering by tissue or disease category but not necessarily by the age-of-onset profiles. Genes with consistently increased mutational load across multiple tissues were found to be associated with autophagy, DNA damage repair, and some immune response categories. Although the functional associations we identified for late-onset diseases do not overlap with these, the interaction between the identified functions and those found to be associated with mutational load could still play a role. Another potential contributor is the change in gene or protein expression. Age-related changes in the transcriptome and proteome are not linear and around 20 and 60 years of age, include changes in the direction or rate [@Anisimova2020; @Colantuoni2011; @Donertas2017]. Similarly, a recent study investigating the changes in proteome after the age of 20, suggested three groups of proteins with changes at 34, 60, and 78 years of age [@Lehallier2019]. They further compared these groups to genes associated with Alzheimer's and cardiovascular disease. The authors showed Alzheimer's had a significant association with only the proteins that change at later ages (60 and 78). However, cardiovascular disease was also associated with proteins that change at 34, and this could potentially explain an earlier age-of-onset. Similarly, the genes defining cluster 1 and cluster 2 genes may have different gene or protein expression trajectories due to various environmental or intrinsic factors. .

### Limitations

We employed one of the largest comprehensive available datasets, including medical and genotype data for almost half a million participants, to perform GWAS on human diseases. However, our study is not without limitations. 

* We used the same cohort to perform multiple GWAS and thus, data is not independent. In our genetic analysis, we correct for disease co-occurrence values which consider if the same people have the diseases, however, it is still possible that the effect is not fully neutralised and we detect more commonality between diseases that co-occur together. This, however, is not likely to cause misinterpretations as most of the late-onset diseases do co-occur and the results still could have biological relevance. 
* In this study, we only used self-reported diseases. Throughout the study, there were several instances that revealed reporting bias. For example, seemingly unrelated diseases such as fractures and appendicitis had high co-occurrence values. Another example is the peaks in age-of-onset distributions of diseases at every 5 years. These both suggest the reporting is not perfectly accurate. 
* We used the UKBB dataset, which has samples mostly from British descendants and thus our results should be validated in independent cohorts. 
* In order to discover the common genetic component between diseases, we followed a GWAS approach and thus were limited to common variants. Another related outcome is that we only analysed common and complex diseases. These limit our ability to detect truly causal variants and instead we detect incremental contributions to risk of disease development. In general, considering that ageing is a gradual and subtle phenotype and most of the late-onset diseases are common, this approach may not prevent the analysis of many interesting diseases for ageing research. However, especially the analysis for evolutionary theories of ageing could benefit inclusion of rare variants and diseases. Moreover, it is important to note that complex diseases generally have strong interactions with environmental factors. However, we only included BMI as a covariate in our GWAS model. Especially studies aiming to find more causal genetic factors influencing multiple diseases should model environmental factors.
* In this study, we had a limited age range, covering individuals up to 65 years old and thus, could not analyse diseases of later ages, such as Alzheimer's Disease. 
* In order to calculate the genetic similarities between diseases, we used a simple overlap-based measure. The main reason was to specifically test the overlapping variants so that we can further analyse that specific set of variants for ageing-relevance, functional associations, and evolutionary characteristics. However, this method is limited as it only considers 'significant' variants and ignores potential association between diseases that have limited sample size or more complex nature due to power issues. Future studies aiming to discover the associations between diseases may use probabilistic approaches to calculate disease-disease similarities that can also consider the factors we considered in this study, such as disease categories and co-occurrences. 
* Lastly, the UKBB only includes genotypes and lack other 'omics data types. Also, although participants provide self-reported data and some of the biomarkers multiple times, they are genotyped for only once and the genetic profile is assumed to represent germline variants. As most of the age-related diseases are influenced by environment, especially to drive conclusions about the causal nature of the identified variants, we would need data spanning somatic mutations, transcriptome and epigenome. 

Future cohorts with a broader age range and data spanning different 'omics data, somatic mutations, health outcomes, and lifestyle information, can enable a better understanding of the genetic mechanisms of age-of-onset determination and establishing the causal link with candidate genes.

### Conclusion

In this study, we have taken the first steps in understanding the relationship between ageing and diseases through a comprehensive analysis of disease age-of-onset profiles and genetic associations in the UK population. Diseases with more similar age-of-onset profile were genetically more similar, and this similarity was not explained by disease categories, co-occurrences, or cause-effect relation between diseases, suggesting biological pleiotropy and common aetiology. The shared genetic component between ARDs overlapped with some ageing-related genes, but diseases with different age-of-onset profiles had different functional associations. We also identified drugs targeting the common genetic component between ARDs, which could target multiple diseases at once and thus alleviate the effects of polypharmacy in the elderly. Lastly, variants associated with ARDs that start to occur at different ages had different evolutionary characteristics, supporting the mutation accumulation and antagonistic pleiotropy theories of ageing.
