# Drug repurposing for ageing {#drugChapter}

\small
**Declaration**

*This work includes results that I have published as a research article in Aging Cell [@Donertas2018]. All the results presented in this chapter are products of my own work. I have conceived the study and performed all the analysis presented here. I must acknowledge the contributions of Matias Fuentealba, Linda Partridge, and my supervisor, Janet M Thornton. Their suggestions helped design this work better and improved it significantly. I also benefited from another article we have published as a review article in Trends in Endocrinology and Metabolism [@Donertas2019]. The summary of existing drug repurposing studies in the introduction of this chapter stems from this article, and Matias Fuentealba had a significant contribution in compiling and reviewing these existing studies. The analysis of different studies, investigating their coverage and comparison is a product of my own work, but again benefits from our discussions with Matias. This work was also supervised by Linda Partridge and Janet M Thornton.*

Dönertaş, H. M., Fuentealba Valenzuela, M., Partridge, L., & Thornton, J. M. (2018). Gene expression-based drug repurposing to target aging. Aging Cell, 17(5), e12819.

Dönertaş, H. M.\*, Fuentealba, M.\*, Partridge, L., & Thornton, J. M. (2019). Identifying Potential Ageing-Modulating Drugs In Silico. Trends in Endocrinology and Metabolism: TEM, 30(2), 118–131.
\* denotes an equal contribution

**Data Availability**

*Supplementary data is available as a BioStudies entry [S-BSST330](https://www.ebi.ac.uk/biostudies/studies/S-BSST330). Data tables are referred using the corresponding file names throughout the text.*
\normalsize

\newpage
## Introduction

Life expectancy has increased steadily in many countries worldwide and is projected to continue [@Oeppen2002; @Kontis2017]. Since ageing is the major risk factor for multiple pathologies, including cardiovascular diseases, neurodegenerative disorders, and cancer [@Niccoli2012], finding interventions that can increase health during ageing is of importance. 

### The malleability of ageing

Since ageing is the major risk factor for poor functioning and disease, intervening to ameliorate its effects could also prevent multiple age-related conditions simultaneously. There is growing evidence for the feasibility of this approach. People who die when they are very old (100, 105, 110) show progressively less multimorbidity at the end of their lives [@Christensen2008; @Andersen2012]. Thus, a healthy ageing phenotype in humans can be achieved, and if we could understand the mechanisms leading to it, we might be able to extend it to the general population. Moreover, work over the past 20 years has shown that lifespan of laboratory model organisms can be greatly extended by genetic and environmental interventions, which also improve health and function during ageing [@Clancy2001; @Lucanic2013; @Xiao2013]. 

Dietary restriction (DR), a reduction in food intake that avoids malnutrition, can extend lifespan and induce a marked improvement in health during ageing in diverse organisms, including rodents [@Fontana2015; @Kapahi2017]. Two studies of rhesus monkeys subjected to DR found that the animals had lowered plasma triglycerides, diabetes, cardiovascular disease, sarcopenia, incidence of neoplasms, and brain atrophy, all features of ageing in humans  [@Mattison2012; @Colman2014; @Vaughan2017]. However, compliance with DR regimes in humans is low, and for this reason, it is not a practical public health intervention.

Changes in diet are monitored by many nutrient-sensing systems, including the insulin / insulin-like growth factor and target of rapamycin (mTOR) signalling network. Many of the interventions further target components of the nutrient-sensing network (NSN), and decrease the activity of IGF / Insulin and / or TOR signalling [@Fontana2010]. This highly conserved network senses nutrients, growth factors, and stress status, and modulates the costly activities of the organism, such as metabolism, growth, and reproduction, accordingly. Genetic interventions that reduce the activity of the network have proved to extend lifespan in nematode worms, fruit flies, and mice [@Kenyon2010; @Fontana2010; @Pan2017]. These long-lived mutants are protected against many natural pathologies of old age and also those associated with genetic models of age-related diseases. Mechanisms of ageing are highly conserved during evolution, and the process shows a set of characteristic hallmarks of ageing, which are also present in the aetiology of age-related diseases [@Lopez-Otin2013; @Partridge2018]. Interventions that improve health during ageing and increase lifespan in laboratory animals do so by reducing the impact of one or more of these hallmarks.

Pharmacological intervention can also extend animal lifespan. The DrugAge database reports drug-induced lifespan extensions up to 1.5-fold for *C. elegans*, 1.1-fold for *D. melanogaster*, and 31% for *M. musculus* [@Barardo2017]. Some of these chemicals may mimic the effects of DR [@Fontana2010]. For example, resveratrol, which induces a similar gene expression profile to dietary restriction [@Pearson2008], can increase lifespan of mice on a high-calorie diet, although not in mice on a standard diet [@Strong2013]. Rapamycin, directly targets the mTORC1 complex, which plays a central role in nutrient sensing network and has an important role in lifespan extension by DR [@Mair2008]. Rapamycin extends lifespan by affecting autophagy and the activity of the S6 kinase in flies. However, it can further extend the fly lifespan beyond the maximum achieved by DR, suggesting that different mechanisms might be involved [@Bjedov2010]. Nevertheless, the mechanisms of action for most of the drugs are not well known.

### Previous in silico studies to discover anti-ageing drugs {#prevDrugStudies}

Here we review 11 recent in-silico studies aiming to identify and prioritise pro-longevity drugs for animal models and humans. All such studies have been enabled by the development of powerful databases for the annotation and curation of genes/proteins (Ensembl [@Herrero2016], UniProt [@Consortium2017]), their associated functions and pathways (Gene Ontology [@Ashburner2000], KEGG [@Kanehisa2016], Reactome [@Fabregat2018]) and chemical ligands and drugs interacting with them (ChEMBL [@Gaulton2017], DrugBank [@Law2014], STITCH [@Szklarczyk2016], DGIdb [@Griffith2013], PDB [@Berman2000]) or affecting their expression (Connectivity Map [@Lamb2006], CREEDS [@Wang2016]), as well as drugs (DrugAge [@Barardo2017], Geroprotectors.org [@Moskalev2015]) and targets (GenAge [@Tacutu2018], Ageing Clusters [@Blankenburg2018]) implicated in ageing and age-related disease mechanisms.

Although the published studies of drug repurposing to target ageing use different strategies and sources of data, they can be classified into two main categories: methods employing the structural information to predict drugs potentially interacting with proteins already identified as being involved in ageing, and methods based on the similarity between ageing-related drugs or genes based on molecular structure, interactions, pathways or networks.

#### Methods using structural information to find drugs targeting known regulators of lifespan in model organisms {#drugStrMethods}

The concept here is to find drugs which are known to target those genes that have been implicated in ageing. Two studies adopted methods based on the hypothesis that proteins or ligands with similar structures are likely to bind similar ligands or proteins, respectively, to predict drug-target interactions. The first of these [@Snell2016] aimed to identify novel drugs targeting 3 specific temperature sensing proteins implicated in ageing in the rotifer Brachionus manjavacas *(TRP7, S6P, FhBC)*. The authors used a virtual screening software called FINDSITEcomb that combines protein modelling with sophisticated threading approaches to model the target. The pockets in the model are then compared against the pockets in experimentally determined structures of proteins with ligands or modelled structures with known binders. The ligands of the top 100 ranked pockets are then compared against a library of screened ligands and ranked by ligand similarity. The authors screened 1,347 FDA approved drugs in silico and tested four drugs for each target experimentally in the rotifers for their effects on lifespan and healthspan. Five out of the 12 compounds tested significantly increased the rotifers' lifespan. Changes in healthspan, approximated by swimming speed, reproduction and mitochondrial activity, were also observed. In a subsequent study by the same authors [@Snell2018], the number of proteins analysed was expanded to a set of ageing-related genes found in other animal models with orthologues genes in rotifers. This time a total of 94 targets were screened in silico using the FINDSITEcomb software. The top 1% binding compounds for each target were further ranked by their cumulative lifespan extension achieved by genetic interventions into their targets as taken from experimental model organism data, and filtered according to availability and previously predicted side effects [@Zhou2015]. From the 31 drugs experimentally tested in rotifers by two ten-days survival screens, seven drugs were further tested in two whole life survival analyses, two of them resulting in a median lifespan extension of 13-42%. The pro-longevity effect of these drugs was observed even when drug treatment was initiated in middle age.

Another in silico screening study was restricted to a single gene, AMP-activated protein kinase (*AMPK*), whose activation partially mediates the effects on ageing of dietary restriction (DR) [@Mofidifar2018]. To find new molecules to activate AMPK and theoretically mimic DR, @Mofidifar2018 performed virtual screening using molecular docking of 1,908 FDA approved drugs. The interaction between the top-ranked compounds and their targets was then further checked by more detailed molecular dynamics. The study reported 4 compounds with predicted high affinity for AMPK, but these were not tested experimentally.

#### Similarity-based methods to discover drugs targeing ageing {#drugSimMethods}

Using a priori information on known ageing-related genes, pro-longevity drugs or gene expression profiles, several studies have implemented a series of similarity-based approaches to identify novel anti-ageing drugs. 

**Finding drugs that target known ageing-related genes.** Given that drugs targeting ageing-related gene products are expected to affect the ageing process, @Fernandes2016 focused on finding drugs that target human genes whose orthologues in animal models are associated with longevity. The drugs were ranked by the likelihood of targeting ageing-related genes among all targets. For this calculation, only inhibitory drugs interacting with anti-longevity genes and activators targeting pro-longevity genes were considered. In total, 376 drugs were obtained of which 20 were considered to be statistically significant. Thirteen targeted histone deacetylases, and three were previously associated with lifespan extensions in animal models. Recently, a study I have contributed used a composite set of ageing-related genes with direct evidence for influencing human ageing, together with physical and functional drug-protein interactions, to implement a similar gene-set overlap analysis [@Fuentealba2019]. This study also considered other levels of biological actions including pathways, functions and protein-protein interactions. Three of the top 10 compounds ranked highest on an aggregate score were previously shown to increase lifespan in animal models, and seven had been proposed to affect longevity by other drug-repurposing methods. The pro-longevity effects of the top-ranked compound (tanespimycin) was experimentally validated in *Caenorhabditis elegans*.

**Finding drugs similar to known pro-longevity drugs.** An alternative approach is to find drugs similar to known pro-longevity drugs using machine learning, which is a strategy well-suited for prediction tasks. @Liu2016 attempted to predict new pro-longevity drugs for *C. elegans*. They adopted a semi-supervised algorithm trained with high-confidence pro-longevity drugs derived from an experimental screen for *C.elegans* [@Ye2014], together with their associated ageing-related genes curated from the literature and the GenAge database [@Tacutu2018]. They produced a rank-ordered list of 785 drugs with a potential to increase lifespan in worms, with experimental validation for one drug in their list, using a lifespan assay. A separate machine learning approach was trained with chemical descriptors of known pro-longevity drugs, and functional annotation of their targets [@Barardo2017]. Using a supervised algorithm (*i.e.* random forest), they generated a ranked list of drugs predicted as lifespan extending compounds, although no validation was performed. 

**Comparing transcriptome signatures from ageing and drugs.** The Connectivity Map (CMap) Resource provides drug-induced expression profiles for 1,309 compounds (version 2). Comparing these profiles with ageing-related gene expression signatures using a gene-set enrichment analysis can reveal drugs that generate changes in expression correlated (positively or negatively) to those seen in ageing (or any other biological process or disease). This approach requires no a priori list of 'ageing genes' and can therefore potentially identify new targets, based solely on expression profile similarities. The first study used Dietary Restriction (DR) expression profiles in rats and rhesus monkeys to find DR mimetics [@Calvert2016]. They identified 11 drugs that could potentially increase lifespan by mimicking DR. They experimentally tested several of the drugs in *C.elegans* and most extended lifespan. I also follow a similar methodology and instead aim to find drugs that target genes that show the most reproduble age-related changes in gene expression in the human brain. The results of this study is presented under Section \@ref(drugstudy). Importantly using this data-based approach I aim to identify novel drugs and genes, not previously associated with ageing. More recently, after the publication of the work I present in this Chapter, @Yang2018 used a network-based methodology, called ANDRU (ageing network-based drug discovery). Instead of relying on model organisms, this approach was also driven by human transcriptome data (GTEx) from young and old adipose and artery tissues and signatures from the CREEDS database [@Wang2016] to identify differentially expressed genes within the ageing-related networks and drugs reversing these changes. They report three distinct drugs ranking as the first five. Although none is previously reported as a life-span modulator, these drugs target pathways that change in expression with age, such as metabolic enzymes and lipid metabolism.

#### Approaches to prioritise drugs for testing

One of the major challenges to developing anti-ageing drugs is experimental validation. Since clinical trials involve many ethical considerations and are very expensive, such drugs are pre-tested in model organisms. In this spirit, @Aliper2016, aimed to predict which pro-longevity drugs previously tested in *C.elegans* could work in humans. Using young and old human stem cell expression profiles and an algorithm called Geroscope that maps the gene expression changes with age to ageing-related signalling pathways, they ranked a set of candidate drugs by their likelihood of targeting these pathways. To do this they calculate the pathway activation strength (PAS) for each drug. They shortlisted ten compounds with pro-longevity effects in *C. elegans*, and tested six of them for geroprotective effects in senescent human fibroblast cultures. While the majority of tested drugs improved senescence-associated phenotypes, one drug (PD-98059), a highly selective MEK1 inhibitor, also showed life-prolongeing and rejuvenating effects.

Comparably, to assess which 'human' drugs and chemicals are likely to modulate the *C. elegans* and *Drosophila melanogaster* orthologue of the target, @Ziehm2017 developed a method to rank chemicals binding to genes implicated in human ageing. They generated an empirical scoring function that considers the conservation of the domain and binding site at the sequence level between the animal and the human protein, predicted binding energy for the compounds for the human targets and experimental bioavailability, in addition to scores for drug-likeness, promiscuity, purchasability and development status. Although the authors provided no experimental validation, they conducted a comprehensive literature mining and molecular docking procedure to validate their results. 

### Research objectives

Although previous studies tried to discover drugs that can affect ageing, they all focus on genes or drugs related to lifespan regulation. The role of these drugs in promoting healthy ageing in humans is still an open question. In this study, using gene expression data for human brain ageing, we aimed to discover not only new pro-longevity drugs but also those that can improve health during ageing. Human brain undergoes substantial structural changes with age, including changes in brain weight, white and grey matter volumes. Accompanied by the altered intercellular communication and synaptic loss, these changes bring about cognitive decline, neurodegeneration and memory loss [@Salthouse2009]. The biological processes showing a change in expression include pathways related to synaptic and cognitive functions as well as proteostasis [@Lu2004], suggesting gene expression changes in the ageing brain could be used as a surrogate to find drugs to target detrimental effects.

Here, we extended the previous approaches to identification of new anti-ageing drugs for humans, by focusing directly on human ageing data. We used a framework that does not require any prior knowledge and is thus robust to biases in the literature and databases on ageing. Moreover, using human age-series data, this methodology has the potential to discover drugs affecting both life- and health-span. Through a meta-analysis of multiple gene expression datasets, we aimed to first compile a robust signature that can characterise ageing in the human brain. We then aimed  to identify a list of potential drug candidates that could influence human brain ageing, using drug-induced RNA expression profiles deposited in the Connectivity Map (CMap) [@Lamb2006]. We also assess the performance of our method in relation to previous knowledge and other computational studies.

## Gene expression-based drug repurposing to target ageing {#drugstudy}
### Analysis of age-related changes in RNA expression in human brains {#ArrayChangesAgeing}

We analysed data from seven, published, microarray-based studies of age-related changes in RNA expression [@Barnes2011; @Berchtold2008; @Colantuoni2011; @Kang2011; @Lu2004; @Maycox2009; @Somel2010; @Somel2011]. The data came from 22 different brain regions, and the ages of the donors ranged from 20 to 106 years (Figures \@ref(fig:drugFig1)a, \@ref(fig:drugFigS1)). The data for each brain region in each study were analysed separately, resulting in 26 datasets.

```{r drugFig1,out.width = '100%', fig.cap=c("a) Age distribution of the brains from which the data sets used in the study were derived. The error bars show the standard deviation of the sample frequency for different brain regions in data sources with multiple brain regions. b) Hypothetical gene expression plots, demonstrating how Spearman's correlation coefficient and p-value behave when the association is weak or nonmonotonic. c) Pairwise Spearman's rank correlation coefficients across data sets. The intensity of the colours on the heatmap shows the magnitude of the correlation coefficient"), fig.scap="a) Age distribution of the samples. b) Hypothetical gene expression plots demonstrating the method. c) Pairwise correlation plot across datasets."}
knitr::include_graphics(path = "figures/drug/Figure1.png")
```

To characterise the association between the gene expression and age, we calculated the Spearman's correlation between the expression level and age, for each gene, in each dataset separately. We first calculated the number of significant changes (FDR corrected p<0.05) in each dataset (Figure \@ref(fig:drugFigS2)). While there were two datasets with a large number of significant changes, most of the datasets did not show substantial significant change. This can be explained by several factors, most importantly i) most of the datasets had a small sample size, providing insufficient power to detect changes in most of the cases, and ii) Spearman's correlation test calculates significant monotonic changes, whereas it is likely that many of the changes are not exclusively monotonic throughout ageing. Thus, we applied another approach, using the correlation coefficient to capture significant trends across datasets, instead of within a dataset (see Section \@ref(ageingSignatureMethods)). While the p-value is affected by the number of the samples and the strength of the monotonic relationship (Figure \@ref(fig:drugFig1)b), the sign of the correlation coefficient can be used to capture consistent trends of up- or down-regulation once coupled with an appropriate testing scheme. This strategy requires the datasets to be concordant and reflect genuine age-related changes. We first investigated if this assumption was valid. To assess the concordance among datasets, we used Spearman's correlation coefficients and calculated the correlation between expression-age correlations between datasets (Figure \@ref(fig:drugFig1)c). We observed a weak correlation with a median pairwise correlation coefficient of 0.29. To calculate the significance of this correlation, we developed a stringent permutation scheme specifically designed to account for the dependence between genes as well as the datasets (see Section \@ref(drugPermutationTest)). We concluded that a median correlation coefficient of 0.09 would be expected by chance and that our observation (median $\rho$=0.29), is statistically significant (p<0.001). Based on these correlations, datasets clustered according to the data source rather than to the brain region. This observation is in line with the previous studies suggesting that ageing-related changes are small and heterogeneous, making them difficult to detect [@Somel2006]. We therefore tested for significant correlations across datasets from different studies. When we excluded the correlation coefficients among the datasets generated by the same studies, we still observed a significant correlation coefficient of 0.22 (permutation test p<0.001, $\rho$=-0.002 would be expected by chance), showing that we have significant correlations among different data sources as well. Using these correlations, we proceeded to compile the ageing-signatures, reflecting consistent trends.

### Defining the ageing signature {#ArrayAgeingSignatureResult}

To construct a robust ageing signature, we identified the age-related changes that were observed across all datasets, irrespective of the effect size. We thus focused on global age-related changes in the brain, rather than region-specific changes, and the set of genes that showed gene expression changes in the same direction across all datasets (Figure \@ref(fig:drugFig2)a).

```{r drugFig2,out.width = '60%', fig.cap=c("Method summary for (a) compiling the ageing signature and (b) theCMap algorithm."), fig.scap = "Summary of the methods."}
include_graphics(path = "figures/drug/Figure2.png")
```

This profile consisted of only 100 up- and 117 down-regulated genes (Supplementary Table S2, Figures \@ref(fig:drugFigS3), \@ref(fig:drugFigS4)), 'the ageing signature'. To establish the robustness of the ageing signature, we calculated the statistical significance of the number of consistent changes with the same permutation scheme used to test the correlations among datasets. This methodology randomizes the age of each individual, making it possible to test the null hypothesis where there is no association between expression and age while retaining the dependence between genes and datasets (see Section \@ref(drugPermutationTest) for details). The number of consistent expression changes across brain regions was significant (p<0.001, Figure \@ref(fig:drugFigS6)a,b), establishing that the ageing signature indeed has biological meaning.

To further test the robustness of the ageing signature, we used an independent data set, consisting of gene expression in human brain generated by the GTEx Consortium [@GTEx_Consortium2015], consisting of data from 99 individuals, 13 brain regions and ages between 20-79 (Figure \@ref(fig:drugFigS3)b, Supplementary Table S1). These data were generated using RNA-Seq, allowing us to assess the robustness of the ageing signature to different technology platforms. We used pipeline previously applied to the microarray data to calculated age-related expression changes for each gene in each brain region separately.  The pairwise correlations between the GTEx datasets were higher than with the other dataset, and they tended to cluster together (Figure \@ref(fig:drugFigS5)). We found 1189 up- and 1352 down-regulated genes that showed the same direction of change across all GTEx brain regions (Supplementary Table S2), compared with only 100 and 117 in the microarray ageing signature. A likely explanation is that samples from different brain regions from the same individuals were used in GTEx, while the microarray ageing signature combined seven independent studies and different microarray platforms. The numbers of shared expression changes based on permutations were 127 and 131.5, for down- and up-regulated genes, suggesting a higher false positive rate in the GTEx dataset. Nevertheless, the numbers of consistent up- and down-regulated genes in the GTEx dataset were also significant (p=0.001, Figure \@ref(fig:drugFigS6)c-d). The numbers of common up- and down-regulated genes across the GTEx and microarray signatures were 50 and 48, respectively, both statistically significant (binomial test p < 2.2e-16 for both), demonstrating that the ageing signature was reproducible.

### Biological processes associated with the ageing signature {#ageingSignatureBiology}

We next investigated the biological processes associated with the microarray ageing signature. Using the genes that were consistently expressed in all data sources as background, we did Gene Ontology enrichment tests for consistently up- and down-regulated genes, separately (Figure \@ref(fig:drugFig3), Supplementary Tables S3-4). Down-regulated genes were enriched in synaptic functions and biosynthetic processes (FDR corrected p<0.05), while differentiation and proliferation-related categories showed enrichment for the up-regulated genes (FDR corrected p<0.05). These results are consistent with the findings of earlier brain ageing transcriptome studies [@Lu2004; @Naumova2012; @Xue2007]. Oddly, ossification-related biological processes also showed significant enrichment for the up-regulated genes. However, except for one gene, these ossification-related categories shared all genes with the more generic development-related categories. Thus, this result could be interpreted as a general up-regulation of the development-related processes rather than ossification-related categories.

```{r drugFig3,out.width = '100%', fig.cap=c("Gene Ontology Biological Process Categories significantly enriched in (a) down- and (b) upregulated genes in the microarray ageing signature. Red circles represent the genes, and diamonds show the significantly associated GO categories, where FDR adjusted p < 0.05. The size of the diamonds represents the effect size (odds ratio)."), fig.scap = "GO Enrichment results for the ageing signature."}
include_graphics(path = "figures/drug/Figure3.png")
```

We repeated the enrichment analysis using the GTEx ageing signature and found 194 and 256 GO BP categories as significantly associated with down- and up-regulated genes, respectively (Supplementary Tables S7-8). Since the number of genes in the GTEx signature is higher, we had more power to detect smaller changes and thus had a higher number of significant associations. However, the effect sizes (odds ratios) for each GO BP category calculated for microarray and the GTEx ageing signature were correlated (Figure \@ref(fig:drugFigS7)). Correlations between the odds ratios calculated for all of the GO categories calculated in both methods were 0.46 and 0.37, for the enrichment in the down- and up-regulated genes, respectively. Correlations increase when we considered only the GO categories that are significantly associated with at least one of the ageing signatures; 0.55 and 0.60, for the enrichment in the down- and up-regulated genes, respectively. This further shows that the ageing signatures are robust. The categories enriched in down-regulated genes included biological processes related to neuronal and synaptic functions, autophagy, post-translational modifications, and translation (see Supplementary Table S7 for the full list). Processes related to response pathways, immune response, macromolecule organisation and lipid metabolism showed enrichment in up-regulated genes (see Supplementary Table S8 for the full list). Interestingly, categories related to ossification were also among the GO categories significantly associated with up-regulation, based on GTEx data.

### Mapping the ageing signature onto drug-perturbed expression profiles

The Connectivity Map (CMap) is a database of drug perturbed gene expression profiles [@Lamb2006]. It consists of 6100 gene expression profiles for 1309 drug perturbation experiments performed on five different cell lines. The CMap algorithm uses a modified Kolmogorov-Smirnov test statistic to calculate the similarity of a drug-perturbed expression profile to the gene expression profile used to query the database. A positive similarity score means that the drug-perturbed expression profile is similar to the query, whereas a negative score indicates a negative correlation (Figure \@ref(fig:drugFig2)b). Based on the random permutations, the statistical significance of the similarity score for each drug is calculated. Thus, the p-value shows the probability of finding the same association when a random signature is supplied. We queried the CMap database and identified drugs that showed significant associations in either direction with the ageing signatures. To determine the robustness of this procedure, we queried the CMap data using the microarray ageing signature, and the top 500 up- and 500 down-regulated genes from the GTEx ageing signature (see Section \@ref(drugCMapMethod)). The correlation was significant (r=0.52, p<2.2e-16, Figure \@ref(fig:drugFigS8)a) showing that the two ageing signatures produce reproducible overlaps with the CMap database.  In order to test the reproducibility and not bias the results due to the technology used to generate the data, we preferred not to combine ageing signatures but report the resulting drug hits from the two signatures separately. Nevertheless, it is noteworthy that the drug similarity scores, generated using the overlap between signatures, show significant correlation with the lists generated using both microarray and GTEx signatures (Figure \@ref(fig:drugFigS12)).

```{r drugFig4,out.width = '50%', fig.cap=c("Similarity score table for the drugs having at least one significant association with the ageing signatures. Each row corresponds to a drug and columns correspond to two independent ageing signatures—using the microarray and the GTEx data sets. The size of score labels indicates the significance of the results (FDR-corrected p < 0.05). The row labels written in bold indicates the drugs in the DrugAge database."), fig.scap = "Drug-Ageing signature similarity scores."}
include_graphics(path = "figures/drug/Figure4.png")
```

Querying the CMap database, we identified 13 drugs significantly associated (FDR corrected p<0.05) with the microarray ageing signature (Table \@ref(tab:drugTable1), Figure \@ref(fig:drugFig4)). Four of these drugs were previously shown to extend lifespan in worms or flies in at least one experiment (Supplementary Table S9). The number of pro-longevity drugs re-discovered using this methodology was statistically significant (p=0.004), and only one drug would be expected based on 10,000 random permutations of drugs. Repeating the same analysis with the GTEx ageing signature, we identified 18 drugs, seven of which were in common with the microarray results, including the four known pro-longevity drugs. In total, 24 drugs were significantly associated with at least one of the ageing signatures. The correlation between the drug similarity scores for these 24 drugs calculated based on the microarray and GTEx data was 0.88 (p<9.44e-09, Figure \@ref(fig:drugFigS8)b), indicating high concordance. Since the similarity scores show high correlation, the rest of the results will be presented for the 24 drugs that are associated with at least one of the ageing signatures.

```{r drugTable1}
read_csv('tables/drug/Table1.csv') %>%
    kable(format = 'latex', booktabs = T,  caption = 'The drugs that are significantly associated (FDR corrected p<0.05) with at least one of the ageing signatures. Drug names denoted with (*) shows the drugs in DrugAge database. "Score" is the mean similarity score given in the CMap output, based on KS test. The similarity scores denoted with (*) show the significant associations. The list is ordered by the mean of the similarity scores from negative to positive. Target or mechanism of action is manually curated from literature or extracted from CHEMBL, DrugBank, and PubChem databases. The targets denoted with (*) are found in the GenAge model organism or GenAge human databases.', caption.short="Drugs that are significantly associated with the ageing signatures.", longtable=T) %>%
    column_spec(4, width = "6cm") %>%
    kable_styling(latex_options = c("striped"))
```

Overall, the method re-discovered seven known pro-longevity drugs in DrugAge database (p=0.00023, based on 100,000 random permutations); resveratrol, LY-294002, wortmannin, sirolimus (also known as rapamycin), trichostatin A, levothyroxine sodium, and geldanamycin (Supplementary Table S9).
 
### Targets of the drugs

Next, we investigated the targets of these 24 drugs, using the ChEMBL, PubChem and DrugBank databases as well as through manual curation of the literature (Table \@ref(tab:drugTable1)), and whether these targets were previously implicated in ageing, using GenAge human and model organism databases (Figure \@ref(fig:drugFig5)). Except for four (rifabutin, securinine, thioridazine, trifluoperazine); all drugs or their target genes had been previously implicated in ageing. Moreover, the drug-target association network showed several clusters with multiple drugs sharing the same targets: i) quinostatin was in the same cluster with two known pro-longevity drugs, wortmannin and LY-294002, targeting PI3K subunits, ii) tanespimycin and alvespimycin shared the same target with another DrugAge drug, geldanamycin, targeting HSP90, iii) vorinostat shared one of its targets, HDAC6, with trichostatin A, another DrugAge drug, iv) thioridazine and trifluoperazine had dopamine and serotonin receptors as targets and v) irinotecan and camptothecin shared TOP1 as their target. The fact that drugs targeting the same proteins / acting through the same mechanism had similar CMap similarity scores (Figure \@ref(fig:drugFig4)) further shows that our results are biologically relevant and reflects potential mechanisms to target ageing.

```{r drugFig5,out.width = '100%', fig.cap=c("Schematic representation of the drug–target associations as a network. Blue and red nodes show drugs and targets, respectively. The drugs with a light blue background are present in DrugAge database and the targets with a pink background are in either GenAge model organism or GenAge human databases."), fig.scap = "Drug-target network."}
include_graphics(path = "figures/drug/Figure5.png")
```

### Drugs can act both by reversing ageing effects and mimicking responses

The general expectation from an 'omics-based drug repurposing study is the identification of drugs that can reverse the abnormalities detected in the disease state *i.e.* identification of drugs with negative similarity scores [@Duran-Frigola2017]. Following the same logic, one might expect drugs with anti-ageing potential to have negative scores. Interestingly, some of the known pro-longevity drugs had positive similarity scores to the ageing signatures, meaning that the drug-induced profile was similar to the ageing signature. A plausible explanation for this observation is that ageing signatures may partly reflect cellular defense responses, helping to alleviate the damaging effects of ageing.

### Characterising the biological functions associated with pro-longevity drugs

In order to identify the biological processes associated with the changes that were reversed or mimicked by the pro-longevity drugs, we used the drugs documented in DrugAge, that were re-discovered in our analysis. We grouped the microarray ageing signature into five categories, based on the expression changes in ageing (up or down), and the pro-longevity drug-induced profile (up, down or inconsistent) (Supplementary Table S5). To compile the pro-longevity drug profile, for each probe-set in the microarray ageing signature, we asked if the seven DrugAge drugs induced similar changes. If the same direction of change was induced by more than half of these DrugAge drugs, then we included these changes in the pro-longevity drug profile (see Section \@ref(drugProExpProf) for the details). We then analysed the biological processes associated with the genes in these categories. The number of genes is small, with no significant changes after multiple test correction. We therefore report the associations based on the highest odds ratios only. For genes down-regulated in ageing, the changes mimicked by the drugs were associated with autophagy and metabolic processes (Supplementary Table S6), while for up-regulated genes, pro-longevity drugs tended to mimic the changes in protein complex / cellular complex assembly-related functions and to reverse the changes observed in protein localisation and immune-related functions (Supplementary Table S6). These findings are consistent with the mechanism of action for the most well-known pro-longevity drugs. For example, sirolimus (rapamycin) is an immunosuppressant approved for human use, and similar drugs can enhance the response of elderly humans to immunization against influenza [@Mannick2014].

### Similarity among significant drugs based on the expression changes at the functional level

In order to analyse the similarities among drugs based on expression level changes, we performed a gene-set enrichment analysis (GSEA) for the drug-induced expression profiles, including all genes irrespective of whether a given gene is in the ageing signature (see Section \@ref(drugHitGO)). To measure the similarity between drugs, we calculate the Spearman's rank correlation coefficients between the enrichment scores and then cluster drugs based on these correlation coefficients. Notably, drugs targeting the same proteins or pathways, *e.g.* PI3K inhibitors LY294002, wortmannin and quinostatin, clustered together. Using this functional level approach, we grouped drugs into four groups: i) known pro-longevity drugs, ii) drugs clustering together with at least one pro-longevity drug, iii) drugs which clustered together but did not cluster with any known pro-longevity drugs, and iv) drugs which did not cluster with any other drugs (Figure \@ref(fig:drugFigS10)).

### Ageing signature in other tissues

Since our analysis is based on an ageing signature compiled using only the brain tissue, we also explored if this signature is representative of the other tissues. A plausible way to approach this question is repeating the same analysis using other tissues. However, it is not straightforward because i) the number of datasets available for the other tissues limits the capacity of our approach to compile consistent signatures, increasing false positives, and ii) we find that the ageing-related changes in other tissues are not as consistent as in brain (Figure \@ref(fig:drugFigS11)a). Thus, we choose another approach and asked if the direction of change for the ageing signature we compiled is similar to the direction of change in other tissues (Figure \@ref(fig:drugFigS11)c). We also tested the significance of the similarity in the direction of change based on random permutations. As expected, GTEx brain data showed the highest percent similarity to the array signature. 8/35 datasets showed more dissimilarity for the down-regulated genes (*i.e.* percent similarity was lower than 50%), while only two were statistically significant, namely, liver and atrial appendage. Similarly, only 6/35 datasets showed more dissimilarity for the up-regulated genes, while none was significant. We repeated the analysis with the GTEx signature and observed similar results with only exception that there were five datasets with significant dissimilarity for the down-regulated genes (Figure \@ref(fig:drugFigS11)e). Thus, it is possible that brain signature includes some brain-specific changes but based on significant similarity, we can say it is also representative of other tissues.

## Comparing the results with other eleven in silico studies

My study and all the other studies described in the Section \@ref(prevDrugStudies) had different aims, methods, and data sources. To facilitate their comparison, we have summarised each study in terms of i) the drugs identified, ii) the genes targeted by these drugs, and iii) biological pathways (KEGG) in which these genes are involved (Figure \@ref(fig:temrevFig2)).  Additionally, we compared the results with the manually curated databases of ageing-related genes (GenAge) and drugs (DrugAge). 

**Drugs:** Overall only 12% of all DrugAge drugs are prioritised by at least one study (41 of 346 drugs in DrugAge), with one in every four drugs discovered already present in DrugAge, reflecting the prioritisation process and the low number of drugs reported as significant by each study (15 drugs on average). In addition, the 163 drugs identified usually differ between studies with 91% (149 drugs) of them being specific to one study. From the remaining 14 drugs present in more than one study, trichostatin, geldanamycin, tanespimycin and vorinostat were identified by three studies (Figure \@ref(fig:temrevFig2)a) and, while only the first two are present in the DrugAge database, the remaining two have been experimentally validated for pro-longevity effects in animal models [@Fuentealba2019; @McDonald2013]. Most studies resulted in a list of drugs containing mainly novel candidates not present in DrugAge (122 drugs were classified as novel discovery), the only exception being @Aliper2016, which focused only on a set of known pro-longevity drugs. We also note that 66% of the 122 drugs (*i.e.* 81 drugs) known to target ageing-related proteins were prioritised by the computational studies reviewed above, as expected considering that these drugs are included in some of the databases used by some of the methods during the prioritisation process.

```{r temrevFig2,out.width = '90%', fig.cap=c("Drugs, human genes and KEGG pathways discovered in the 12 studies. Circular heatmap of the drugs discovered by each of the 12 studies (drugs sector), genes targeted by these drugs (human genes sector), and the pathways including these genes (KEGG pathways sector). Drugs, genes and pathways are clustered independently to reflect discovery patterns from the studies. For the drugs and human genes sectors, the inner circle shows whether drugs or genes were previously associated with ageing, based on the DrugAge or GenAge database, respectively. If a drug was not present in DrugAge, it was classified as “candidate”, and the cell was coloured blue, whereas if the drug was already in DrugAge, it was classified as “previously discovered” and the cell coloured in orange. An equivalent strategy using the GenAge databases instead of DrugAge was used for the human gene sector. In the inner wheel we present the overlap with drugs targeting ageing-related genes (drug sector – GenAge Human/Model tracks) and for the human gene sector the overlap with genes targeted by the drugs in DrugAge (Human genes sector – DrugAge track). The KEGG pathways sector shows the proportion of genes on each pathway targeted by the drugs discovered by each study. The cells representing KEGG pathways were coloured using a continuous gradient from white to green, where white means that none of the genes in that pathway were targeted by the drugs identified. In the section closer to the centre of the heatmap, we also showed the proportion of ageing-related genes in these pathways as well as the coverage of genes targeted by drugs in the DrugAge database."), fig.scap = "Drugs, human genes and KEGG pathways discovered in the published drug-repurposing studies."}
include_graphics(path = "figures/drug/temFigure2.png")
```

<!-- ```{r temrevFig3,out.width = '80%', fig.cap=c("Candidate Drugs and genes from the Druggable genome proposed by multiple studies. a) Network representation of candidate drugs discovered by multiple studies and the studies in which they were found. Orange nodes show drugs previously discovered to affect lifespan in animal models (DrugAge) and blue the novel candidates. The edges link drugs identified to the relevant study. b) Distribution of the number of genes targeted by the drugs identified with respect to the number of studies. The x- and y-axes show the number of studies and genes, respectively. The bar above zero studies represents the number of genes not targeted by the novel candidates. The pie charts show the percentage of genes in GenAge (human database) for each category. The boxed numbers show the total number of genes in each category."), fig.scap = "Candidate Drugs and genes from the Druggable genome proposed by multiple studies."} -->
<!-- include_graphics(path = "figures/drug/temFigure3.png") -->
<!-- ``` -->

**Genes:** Overall, 34% of the GenAge human genes (103 genes) and 10% of the GenAge model organism genes (94 genes) were identified in at least one study, reflecting at least in part the different sizes of the datasets, with more than three times the number of Model Organism genes in GenAge. For clarification, the computational methods identified candidate drugs, which are predicted to modulate ageing, amongst the known drugs, most of which are currently used as therapy for a specific disease. 

Based on the DGIdb database [@Griffith2013], 27% of the druggable genome (*i.e.* 796 genes) is targeted by at least one of the drugs identified in the computational studies (Figure \@ref(fig:temrevFig2)b) and, while few genes were identified in multiple studies, some of them were present in the GenAge database [@Tacutu2018]. Two of these genes *DDIT3* (DNA Damage Inducible Transcript 3) and *ERBB2* (Erb-B2 Receptor Tyrosine Kinase 2) were targeted by the drugs prioritised in eight studies. However, nine studies also identified drugs targeting *BIRC5* (Baculoviral IAP Repeat Containing 5) and *KRAS* (KRAS Proto-Oncogene, GTPase), and ten studies predicted drugs modulating *ABCB1* (ATP Binding Cassette Subfamily B Member 1), which have not previously been related to human ageing. Despite this, genes discovered by multiple studies do not necessarily suggest higher relevance to ageing, and may instead reflect research bias (*e.g.* genes targeted by many drugs because of a role in prevalent disease such as cancer). We also observed that 80% of known pro-longevity drugs (*i.e.* 122 of 152 drugs with known targets) target at least one gene targeted by the candidate geroprotective drugs identified by these twelve computational studies. 

**Pathways:** Intriguingly, among all 319 KEGG pathways, 92% include at least one gene targeted by the drugs identified in the twelve studies. The same tendency was observed for genes in GenAge (83% Model GenAge & 74% Human GenAge), or genes targeted by the DrugAge drugs (88%). While this may suggest ageing is ubiquitous and affects all pathways, another possibility is that genes present in many pathways could be discovered repeatedly because they play a central role in diseases and regulatory mechanisms. Although this may not conclusively prove that ageing is ubiquitous, the prioritised candidate drugs clearly have a genome-wide effect. 

## Discussion

In this study, using gene expression data, we identified a set of drugs that are likely to modulate ageing in the human brain. Using a meta-analysis approach, we generated a reproducible ageing signature that represents multiple brain regions and is independent of the platform used for the detection of expression. Using the Connectivity Map, we identified drugs highly associated with this ageing signature.  

**Eleven of the drugs have targets previously associated with ageing.** Based on the DrugAge database, seven of these drugs were previously tested on model organisms and prolonged lifespan in at least one experiment. The fact that we successfully re-discovered a statistically significant number of known lifespan modulators, without using any prior drug ageing information, suggests that the other drugs that we identified also have a high potential to be modulators of the ageing process / lifespan. Vorinostat is a histone deacetylase (HDAC) inhibitor used for the treatment of cutaneous T-cell lymphoma [@vorinostat]. Although not reported in the DrugAge database, vorinostat has already been tested on Drosophila for lifespan extension and shown to increase both mortality rate and survival when the drug is given during 'mid- to late-life' [@McDonald2013]. This drug had the most pro-longevity drug-like profile (Figure \@ref{fig:drugFigS9}, cluster 6) based on our analysis, suggesting that the methodology, as well as the interpretation, yields biologically relevant results. Quinostatin, targeting the catalytic subunit of PI3K, had the highest CMap score with percent similarity above 50% to the pro-longevity drug profile for all four categories (Figure \@ref{fig:drugFigS9}). Considering that drugs targeting PI3K, such as LY-294002 and wortmannin, extend lifespan in worms and flies [@Barardo2017], quinostatin is a strong anti-ageing drug candidate. Alvespimycin and tanespimycin inhibit the heat shock protein HSP90, which is also inhibited by geldanamycin. Heat shock proteins are implicated in ageing based on both experiments on worms and flies [@Tacutu2018] and human expression studies. Protein aggregation and disrupted proteostasis are a hallmark of ageing [@Lopez-Otin2013]. It is thus plausible that increased activity of HSP90 would reverse the effects of ageing by restoring proteostasis, although its downstream effects might result in reverse [@Fuhrmann-Stroissnigg2017; @McClellan2007]. Tretinoin is a retinoic acid receptor (RAR) agonist widely studied for skin [@Mukherjee2006] and brain ageing [@Enderlin1997]. RAR genes in are implicated in synaptic plasticity, learning, memory, and pathological conditions such as Alzheimer's disease [@Lane2005]. GW-8510 is a cyclin-dependent kinase 2 / 5 inhibitor and was suggested to be neuroprotective [@Johnson2005]. 15-d prostaglandin J2 activates PPARG, which shows decreased expression with age, which can be restored by DR [@Tacutu2018]. Camptothecin and irinotecan both target TOP1, which alters the topological state of DNA during transcription and can inhibit Warner syndrome protein (WRN), which functions in DNA repair [@Shamanna2016]. These two drugs, as well as daunorubicin which targets TOP2A, may therefore worsen health status. Cinchonine, which targets CYP2D6 was similar to pro-longevity drug profile in terms of the genes up-regulated by the pro-longevity drugs but show opposite profile for the down-regulated genes. The functions that are associated with the genes down-regulated by the drugs are autophagy or immune function related categories. Most of the known pro-longevity drugs are suggested to function through inhibition of PI3K / mTOR pathways, favouring autophagy. It appears that cinchonine would not function in the same way. However, considering that it targets CYP2D6, which was shown to have a role in lifespan regulation in *C. elegans*, it is possible that this drug has a distinct mechanism to modulate ageing. Mann et al. previously suggested that expression of CYP2D6 increases with age in the human brain and is lower in Parkinson's disease [@Mann2012]. Considering that they suggest this protein might be important to inactivate neurotoxins, inhibiting this protein using cinchonine might function in the same direction and exacerbate ageing by down-regulating one of the cellular responses. 

**New targets and mechanisms to modulate ageing.** Thioridazine and trifluoperazine are serotonin and dopamine receptor antagonists used for the management of psychoses, including schizophrenia. Thioridazine is withdrawn from the market due to its side effects related to cardiac arrhythmias. @Ye2014 screened a library of compounds for lifespan extension in *C. elegans* and identified a couple of drugs targeting serotonin and dopamine receptor antagonists including thioridazine hydrochloride, which extends lifespan by 31% in *C. elegans*. Thus, it is likely that these drugs also have anti-ageing effects. Emetine is the principal alkaloid of the ipecac root. It is a eukaryotic protein translation inhibitor. A recent study investigated the effect of protein translation inhibition on cellular senescence. They suggest that cytoplasmic protein accumulation is an important cause of the cellular senescence and mild protein translation inhibition can prevent senescence induction in normal and tumour-derived human cells [@Takauji2016]. Although both this information and our results suggest that emetine can help alleviating the ageing, @Takauji2016 did test the effect of emetine on senescence and could not detect any significant result. Atropine oxide is predicted to target muscarinic acetylcholine receptors [@CHEMBL2146145], which are suggested to be important for various brain functions as well as pathologies such as Alzheimer's and Parkinson's diseases [@Langmead2008]. The information regarding the effect of atropine oxide on different muscarinic acetylcholine receptors, however, is limited to make a conclusion whether this drug could be beneficial or damaging for the human brain ageing. Securinine is a GABA(A) receptor antagonist. GABA receptors are started to gain attention as potential targets for neurodegenerative diseases [@Rissman2007]. The drugs tested and shown to have an impact on cognitive abilities so far, however, are mainly GABA(A) agonists or GABA(B) antagonists [@Li2016]. Considering that GABA(A) subunit expression levels show a decrease with age (Supplementary Table S2), and securinine is an antagonist, it is possible that it acts in the same direction as ageing and exacerbates it.  Rifabutin shows high similarity to the pro-longevity drug profile and clusters together with levothyroxine sodium and geldanamycin, which are known pro-longevity drugs (Figure \@ref(fig:drugFigS9), cluster 2). Rifabutin is an antibiotic but it is reported to also target BCL6 [@Evans2014]. BCL6 gene is not in GenAge databases, however, there are studies linking this gene to ageing using human gene expression data [@Glass2013] and through its role in cell proliferation and senescence, regulated by miR-127 [@Chen2013]. Thus, it is possible that rifabutin helps to reduce the effect of damaging changes induced by ageing, through targeting BCL6.

**Challenges.** 'Omics-based drug repurposing studies, such as the CMap, aim to identify drugs reversing the profile induced by a biological state of interest. Ageing is a time-dependent, complex phenomenon, which induces subtler changes compared to development [@Donertas2017], or to a disease state such as Alzheimer's [@Avramopoulos2011]. The 'omics profile reflects two potentially distinct contributions: - the detrimental effects which occur with age (*e.g.* accumulation of mutations) and the potentially beneficial responses to those changes (*e.g.* the immune response). As a result, CMap similarity score is not conclusive on its own. In order to characterise the potential effects of drugs on ageing (anti- or pro-ageing drugs), we use three different approaches: i) comparison of the drug-induced expression profiles with the known pro-longevity drug profile (Figure \@ref(fig:drugFigS9)), ii) functional analysis of the drug-induced gene expression changes (Figure \@ref(fig:drugFigS10)), and iii) compilation of literature on the drugs and targets. Based on these analyses we suggest that eight of seventeen drugs (quinostatin, trifluoperazine, thioridazine, vorinostat, alvespimycin, tanespimycin, rifabutin, and 15-d prostaglandin J2), which are not in DrugAge, are likely to have positive effects, whereas, topoisomerase inhibitors (camptothecin, irinotecan, and daunorubicin) can be detrimental and could act as pro-ageing drugs. Four of the remaining drugs, which are cinchonine, securinine, emetine and tretinoin, do not cluster closely with any known pro-longevity drugs in Figure \@ref(fig:drugFigS10). Literature, however, suggests cinchonine and securinine are likely to have negative effects, whereas emetine and tretinoin could act as anti-ageing drugs. GW-8510 and atropine oxide could not be classified because neither the clustering results nor literature evidence are conclusive.

### Limitations

In this study, we applied a data-driven methodology to overcome research bias and increase the chance of discovering novel drugs to target ageing. However, there are several aspects that could limit our study. It is important to note that none of the cell lines used to generate the CMap data originates from the brain. The assumption for using the CMap algorithm is that the effect we see in diverse cell-lines reflects the global profile of the drug perturbation and thus should be also transferable to the brain. However, it is possible that drugs have cell or tissue-specific effects. Even if the drugs induce the same expression changes in brain cells, an important question is: Can they cross the blood-brain barrier to target the brain? If some of these drugs have side effects on the CNS, it might be an indication that these drugs can affect the brain and can be re-purposed to target brain ageing. Only eight of the 24 compounds have reported side effects and all of them has at least one reported effect on the nervous system, based on MedDRA system organ classes (Supplementary Table S10). This implies that these drugs can affect CNS, although we do not have information on their ability to cross the barrier. The rest may or may not cross the barrier to influence the expression in the brain, but they may also improve health by targeting generic changes throughout the body. The ageing signatures from brain tissue show a modest but significant similarity to expression profiles from non-brain tissues (Figure \@ref(fig:drugFigS10)). Thus, it is possible that we identified not only drugs specifically targeting ageing in the brain but also drugs targeting ageing in other tissues. It is also possible that there are drugs which can target brain ageing with more potency, but we cannot identify them because we do not have drug-induced expression profiles for brain cells. Another important technical drawback is that the data we used to generate the ageing signature are bulk RNA expression datasets, where the expression profile is an average of all the cell types in the human brain. Focusing on the changes that are observed ubiquitously across all brain regions, we aimed to focus on global changes which are unlikely to be driven by cell type differences. However, future datasets generated using single-cell expression profiling can greatly improve the understanding of both the ageing process itself and how the interventions work.

### Conclusion

To summarise, this study provides an unbiased identification of drugs that can target human brain ageing. We first compiled a set of gene expression changes that can characterise human brain ageing and asked if there are drugs which alter the expression of the same genes. We identified 24 drugs, seven of which were among known pro-longevity drugs. Our analysis suggests that anti-ageing drugs may act by mimicking the response while it is also possible that they can reverse the detrimental changes in ageing. Based on the literature research, we concluded that some of the drugs we identified can directly modulate the lifespan, whereas some are more likely to function by improving the cognitive functions and promoting the healthy ageing. We are in the process of experimentally testing a group of the drugs that we have identified. We hope the information presented in this study will guide research community to further test and identify chemical modulators of the ageing process in humans.
