# Concluding remarks {#concludingRemarks}

In this thesis, the aim was to approach a better understanding of ageing through computational studies. In particular, age-related change in heterogeneity, the link between ageing and age-related diseases, and drug repurposing for ageing were discussed. In this chapter, I aim to both summarise the major findings and also give a future perspective focusing on each chapter separately. Finally, I will summarise future directions by which computational studies can facilitate research on ageing. 

## Overview and future perspective

### Age-related changes in gene expression heterogeneity

In Chapter \@ref(HeterogeneityChapter), the changes in gene expression heterogeneity during postnatal development and ageing were explored. The main conclusions of this study were:

* Age-related increase in heterogeneity is widespread at the gene-, functional category-, pathway-, and transcriptome-levels. 
* Previous studies focusing on gene expression variability in ageing reported different genes as differentially variable in ageing. This may stem from the differences in the tissues, organisms, and ages that were analysed or could be due to the stochastic nature of increased variability. Using different methodologies on the same dataset, we showed that the preprocessing steps and heterogeneity measures used in different studies also make a significant difference in detecting the 'differentially variable' set of genes. Despite this difference, a widespread increase in heterogeneity was observed with all combinations of preprocessing steps and measures of heterogeneity.
* We also showed that the age-related increase in heterogeneity is a characteristic of post-adult years (20+) but not pre-adulthood (0-20 years of age). 
* Although the increase in heterogeneity is not specific to certain functional categories, or pathways; we found that the genes that become more heterogeneous across all datasets were enriched in many of the longevity-regulating pathways. However, they did not overlap with the exact genes that increase/decrease lifespan in laboratory organisms.
* Finally, we showed that not only specific transcription regulators but also the number of regulators is associated with the increase in heterogeneity during ageing and this is uncoupled from the change in expression during development.

However, some questions remain unanswered and require further experimental and computational studies.

* In this study, we analysed 19 datasets covering 17 brain regions. The brain is a post-mitotic tissue and might include different sources of cellular damage compared to mitotic tissues (e.g. liver) or tissues that are more open to environmental influence (e.g. skin). Thus, future studies comparing different types of tissues can help understand the source of heterogeneity in gene expression.
* We showed an association between the number of regulators and increased heterogeneity. However, both the molecular nature of this association and also whether there are other contributing factors need further exploration. A new study design allowing reliable detection of somatic mutations, and protein levels coupled with the gene expression levels is important to understand the cause of increased heterogeneity during ageing.
* We showed that the reproducible increase in heterogeneity is limited to post-adult ages; however, the reason why there is a difference with postnatal development could not be explored. Likely explanations include active protection from damage or higher tolerance to damage. Moreover, damaging effects may need to accumulate to a certain level to create phenotypic consequences. Understanding the reason would require both a better comprehension of the causes of heterogeneity and a balanced study design with more samples from both development and ageing. 
* In this study, we focused on inter-individual heterogeneity with the analysis of bulk RNA expression levels. However, there are already multiple reports suggesting intra-individual heterogeneity also increases with age [@Davie2018; @Enge2017; @Hernando-Herraez2019; @Martinez-Jimenez2017]. Moreover, different genes also show a difference in their inter-individual variability at an adult age. A comparative analysis of these two phenomena together with our findings could help understand the mechanisms of heterogeneity.

### The link between ageing and age-related diseases

In Chapter \@ref(UKBBChapter), we aimed to understand the link between ageing and age-related diseases through a genome-wide association study of 116 diseases in the UK Biobank. The main conclusions were as follows:

* Diseases with similar age-of-onset profiles show a higher genetic similarity which is not explained by disease categories, co-occurrences, and disease cause-effect relationships.
* Common genetic factors shared between age-dependent diseases significantly overlap with regulators of cellular senescence and the targets of drugs known to modulate lifespan in model organisms. However, the intersection is small.
* We identified biological functions associated with different age-of-onset profiles. Although the two age-dependent clusters both had associations with known ageing-related genes, their functional categories were different. Diseases seen after 40 were associated with lipid metabolism, cell cycle, and cellular response, whereas the diseases seen after 20 were associated with MHC 2 protein binding, fibrinolysis, and regulation of apoptosis. Genes associated with both clusters were enriched among regulators of glucose metabolism, chromatin structure, and transcription. However, in general, genes that are highly pleiotropic and associated with all disease clusters were also regulators of chromatin structure and transcription.
* We checked the risk allele frequencies (AF) of the variants associated with each cluster and found diseases with later age-of-onset had significantly higher AF, which supports both mutation accumulation and antagonistic pleiotropy theories of ageing.
* We searched for drugs that specifically target age-dependent clusters and found several significant hits. 23 of these drugs were already in use for 14 different conditions such as diabetes, hyperlipidaemia, neoplasms, osteoporosis, cardiovascular diseases. This means we can more easily repurpose these drugs to use for the diseases we analysed, which generally co-occur in the elderly. This has the potential to improve health-span and may also alleviate the effects of polypharmacy.

This study, to my knowledge, provides the most comprehensive analysis of the link between ageing and age-related diseases. However, there are multiple aspects that require further investigation:

* In this study, we suggest that there is a common ageing-related genetic factor associated with age-dependent diseases. However, the mechanism by which this factor may influence the age-of-onset profile of diseases could not be explored. Likely explanations include somatic mutations, age-related changes in gene and protein expression levels and epigenome, and gene-environment interactions. We need a significantly improved longitudinal study design including different 'omics and lifestyle data in order to address this question. 
* There is a significant association between the targets of lifespan-modulating drugs and the regulators of cellular senescence. However, we did not detect significant overlap with genes that modulate lifespan in model organisms. One plausible explanation is that these genes are deeply evolutionarily conserved and we do not have common variations that we can detect in this study. However, a detailed analysis of this lack of overlap could improve future study design and also may facilitate knowledge transfer from model organism to human studies.
* In this study, we could only include common diseases because of the sample size limitations. However, a better comparison to analyse evolutionary theories of ageing would include variants that are known to be causal for developmental diseases.
* We suggest some drugs that may improve health-span. The next step would be to test these drugs in model organisms, particularly in mice and rat, and disease-models to measure their effect on health- and lifespan. 

### Drug repurposing for ageing

In Chapter \@ref(drugChapter), we employed an applied study and aimed to identify pharmacological interventions that can improve lifespan in humans. We followed a system-level approach to find drugs that may mimic or reverse the age-related changes in gene expression.

* Through a meta-analyses approach, we first compiled a robust and reproducible list of gene expression changes that characterise ageing in the human brain.
* We compared this signature with the drug-perturbed gene expression profiles in the Connectivity Map database to find drugs that can mimic or reverse these changes.
* We identified 24 candidates, 7 of which were pro-longevity drugs that were previously tested in model organisms. 18 of the 24 had at least one target that was associated with ageing according to GenAge database. We identified 4 completely novel drugs which may provide another angle in ageing interventions. 
* Importantly, we found that some of the pro-longevity drugs act by mimicking the changes in ageing. That means that some of the captured gene expression changes might be beneficial and include responses to the damaging effects.
* We also combined our results with 11 other published studies that perform in-silico drug repurposing to target ageing. Almost 30% of all druggable genome was found to be targeted by at least one drug suggested in these 12 studies. Moreover, 92% of all KEGG pathways had at least one gene targeted by the suggested drugs. These analyses reveal further that ageing is a complex, system-level phenotype.

Rediscovery of a significant number of known pro-longevity drugs suggests our methodology is promising. We also experimentally validated one of the candidate drugs, tanespimycin, in *C.elegans*, as part of another study that I contributed [@Fuentealba2019]. Furthermore, since the input data is from humans it is plausible to suggest that these drugs may provide benefits in humans as well. However, there are several study-specific aspects that may further be explored:

* In this study, mostly because of the limitations in the number of available datasets, we restricted our analysis only to the human brain. However, the whole organism is affected during the ageing process. We show that the ageing signature can also represent changes seen in other tissues. Nevertheless, there might be other drugs that can function in other tissues better, and thus, this study could be expanded to other tissues.
* We found that some of the known pro-longevity drugs we identified mimic the changes observed during human brain ageing. This raises a question about their applicability to humans. Humans live longer than their closely related species and may have adapted some response mechanisms that are not observed in the model organisms, which are short-lived. A comparison of the ageing signature with the age-related changes observed in the model organisms can provide a better understanding.
* Another aspect that we have not addressed in this study is the real-world application of the identified drugs, including factors such as the drug-drug interactions, the age at which drugs could be supplied, the potential effects of long-term use, and bioavailability in humans. Unfortunately, computational studies require more comprehensive experimental datasets that can allow predictive modelling of these factors.
* Lastly, predictive modelling of the overall effect of these drugs on health, health-span and life-span can help prioritisation of the candidates. Today, it is not possible to perform this task for humans as the human clinical trials for ageing interventions are just beginning. However, thanks to DrugAge database, which records the drug, organism, and outcome of the lifespan assays, future studies employing machine learning approaches can prioritise drugs that may have the highest effect on lifespan.

## Future directions in computational biology of ageing

Our understanding of the biology of ageing is improving every day. Theoretical, experimental, clinical, and applied research on ageing has provided significant insight into the mechanisms and interventions to achieve better health in old age. However, so far, these realms of research have been disconnected. The studies presented in this thesis exemplify how computational biology could help to build bridges between different disciplines. However, the potential of computational biology is not limited to the examples I presented. In this section, I summarise the improvements in computational research that can help us improve our understanding of the biology of ageing and provide the toolset to tackle the challenges of age-related decline in health. 

### Single-cell genomics of ageing

Ageing has been studied at the population, organismal, tissue, and cell-type levels. With the advancement of single-cell genomic technology, it is now possible to study ageing even at a single-cell resolution. Studying the molecular changes in single cells within different environments and interactors will provide a more detailed understanding of the mechanisms. This advancement is especially important with regards to the damage-based theories of ageing. Understanding the changes at a single-cell level can suggest better ways of targeting age-related deterioration and help us model age-related changes in specific pathways. 

Our understanding of heterogeneity in ageing can also benefit from single-cell technologies. At the moment, studies approach heterogeneity as a stochastic noise; however, Chapter 2 demonstrates that there is some consistency in increased heterogeneity across datasets. This suggests, there might be a mechanism contributing to the age-related increase in heterogeneity. Studying genetic and epigenetic mutations at the single-cell level can help us understand the source of heterogeneity. 

### Integrative biology of ageing

All the studies I have presented in this thesis further emphasised the importance of having data from multiple layers of information flow (e.g. genome, epigenome, transcriptome, proteome, metabolome). Ageing is a complex phenotype that is far from being driven by just a limited number of genes. By understanding the regulatory changes at multiple levels, we can approach a better intervention strategy. Moreover, it is plausible to argue that the information flow is disrupted at multiple stages during ageing. So far, studies showed an age-related decline in epigenetic stability, which interrupts information flow at a very top level. There are multiple studies focusing on restoring the youthful epigenetic state using drugs targeting epigenetic machinery. However, at the moment we do not know whether the flow of information is disrupted at multiple levels or not. By studying the interactions between different layers of information, we can get a more detailed insight for the source of increased entropy during ageing. 

Integration of phenotype level information such as diseases, frailty, and age-related cognitive decline can further provide a better understanding of the effect of molecular changes on the decline in functional integrity during ageing. Furthermore, combining different types of data, even from different organisms, and cultivating the mechanistic insight provided by the model organisms, can help us approach a molecular and cellular model of ageing.

### Comparative biology of ageing

So far, most of our functional knowledge about ageing originates in model organism studies. However, these model organisms are short-lived and are under captivity for a long time. Whether the mechanisms and interventions found using these organisms are applicable to humans is an important question that awaits exploration. Comparative studies using both genomics and structural data can help the better transfer of knowledge from model organisms to humans. 

